| Software              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Functionality Category(-ies)                                                                                                                                                                                                                                                                                                                        | Functionality Sub-Category                                                                                                                                                                                                                                                                     | Instrument Data Type           | Instrument Data Type Sub-Category | Approaches                              | Computer Skills  | Software Type                         | Interface                                        | Operating System (OS)                                 | Language                                          | Version                | Dependencies                                                                                                                                       | Input Formats - Open                                                                             | Input Formats - Proprietary                                                                                                | Published        | Last Updated       | License                                                                                                                                                    | Website                                                                                                        | Paper                                                              | PMID     |                                                                         |                                             |          |
|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------|-----------------------------------|-----------------------------------------|------------------|---------------------------------------|--------------------------------------------------|-------------------------------------------------------|---------------------------------------------------|------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|------------------|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|----------|-------------------------------------------------------------------------|---------------------------------------------|----------|
| ALEX                  | Analysis of Lipid Experiments (ALEX) is designed for metabolite identification of multiplexed FT MS data. Lipids are identified using the ALEX lipid database containing >20,000 lipid species. This can be searched as part of the GUI or on a web-based tool, which requires a m/z list as input. For calibration of FT MS data, the identity of a well characterised lock-mass ion must be known for the online tool. For the pipeline it can be calculated automatically, using lock mass adjustment. A three-point quadratic interpolation is used for estimating the centroid m/z values of lock mass ions. Lipids can also be quantified using the Orange software.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | DI-MS                          | Orbitrap DI-MS                    | Targeted,Lipidomics                     | Advanced         | Pipeline Python Package       Web App | Command line interface, Web user interface       | Windows XP                                            | C++, Python, SQLite, PHP, Lisp                    | 1                      | MSFileReader 2.2 (Thermo Scientific), Python 2.7, NumPy, PySide, comtypes, SciPy                                                                   | -                                                                                                | Thermo Fisher Scientific .raw                                                                                              | 2013             | 2013               |                                                                                                                                                            | http://mslipidomics.info/contents/?page_id=133                                                                 | http://www.ncbi.nlm.nih.gov/pubmed/24244551                        | 24244551 |                                                                         |                                             |          |
| AStream               | An R package for the annotation of preprocessed LC-MS data (a data.frame of features), detecting isotopes, adducts and fragments. xcmsSet objects can be directly imported into AStream. A quality control step is implemented to identify outlier peaks, which are then discarded. In this peaks below a given intensity are regarded as having 0 intensity and the counts for all samples are transformed into Z-scores, which are used to detect outliers with a high number of 0 intensities. Sample intensity quartiles for probabilities 0.1, 0.2, 0.3, 0.4 and 0.5 are also calculated and standardised as Z-scores. Samples with overly distinct intensity distribution profiles are then also discarded as outliers. The threshold levels for outlier detection can be modified by the user. Signal intensity is then normalised across samples. A pairwise correlation analysis across all features is then performed to identify features which are the same metabolic compound (due to fragmentation). HMDB metabolites are used for compound identification, using monoisotopic mass and biofluid/tissue location. Feature pairs corresponding to carbon isotopes of the same compounds are then identified. Adducts are then identified by searching for peaks within the m/z range of the expected m/z of the adduct.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 2                      | R (ł 3.0.1), multtest, Biobase, plotrix                                                                                                            | data.frame, xcmsSet                                                                              | -                                                                                                                          | 2010             | 2014               | GPL (ł 2)                                                                                                                                                  | http://www.urr.cat/AStream/AStream.html                                                                        | https://www.ncbi.nlm.nih.gov/pubmed/21414990                       | 25426929 |                                                                         |                                             |          |
| BATMAN                | Bayesian automated metabolite analyser for NMR spectra (BATMAN) provides a bayesian model for deconvolution of NMR spectra and an MCMC algorithm to automate metabolite quantification. It is also able to automatically identify metabolites based on user defined metabolite chemical shift lists. A two component model is used for peak deconvolution: for some peaks a metabolite identification can be explicitly defined using an online database (HMDB) and for others the unassigned spectrum is modelled semiparametrically, using wavelets. As well as the R package, there is also the tool splineFitBATMAN, which allows the user to sort the spectra and estimate chemical shifts for each resonance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R, C++, MATLAB                                    | 1.2.1                  | R ł 2.12.1, doSNOW, plotrix                                                                                                                        | ASCII, R                                                                                         | Bruker                                                                                                                     | 2011             | 2014               | GPL-2                                                                                                                                                      | http://batman.r-forge.r-project.org/                                                                           | http://www.ncbi.nlm.nih.gov/pubmed/22635605                        | -        |                                                                         |                                             |          |
| BioSMXpress           | This is an add on to BioSM that is designed specifically to identify endogenous mammalian metabolites based on their molecular structure, as opposed to synthetic molecules. It aims to use the smallest number of structure comparisons possible for the identification of metabolites. 'Scaffolds' (1564 endogenous mammalian compounds identified from KEGG) are searched against for identification - if the smaller structure is an exact substructure of the larger this is considered a match. BioSMXpress only identifies metabolites as biological or non-biological. A list of SMILES is required as input into the software.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                |                                |                                   |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | Java                                              | 1                      | ł Java 1.6, JRE                                                                                                                                    | SMILES                                                                                           | -                                                                                                                          | 2015             | 2015               |                                                                                                                                                            | http://www.mybiosoftware.com/biosmxpress-identification-of-unknown-biochemical-structures-in-metabolomics.html | http://www.ncbi.nlm.nih.gov/pubmed/25859612                        | 20185625 |                                                                         |                                             |          |
| CAMERA                | CAMERA (Collection of Algorithms for MEtabolite pRofile Annotation) is designed as an add-on to XCMS, providing additional grouping based on retention time and chromatographic peak shape and isotopic peaks and adduct annotation. Features are first grouped by their retention times, using 60% of the FWHM around the centroid of the most intense feature that is not yet assigned a compound spectrum. Isotopic peaks are then detected by calculating a pairwise distance matrix and detecting isotopes exhibiting a 1.0033 m/z difference. Chromatographic peak shape is then used to improve spectral separation (pointwise pearson correlation of the intensities between the chromatographic peak boundaries between all pairs of features of a single EIC, pointwise pearson correlation of the intensities between the chromatographic peak boundaries between all pairs of features of all samples and isotope relationship between two features, which are then plotted as scores on a relationship graph using either the Highly-connected-subgraphs or label propagation community algorithms). Adducts, common neutral losses and cluster ions are then annotated using a dynamic rule set, which is then matched against m/z differences. Annotations made in positive and negative mode are then combined and compounds that are detected in both are verified using a novel algorithm and a cross-polarity rule table.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.30.0                 | R (ł 2.1.0), methods, Biobase, xcms (>= 1.13.5), igraph                                                                                            | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2009             | 2016               | GPL (ł 2)                                                                                                                                                  | http://bioconductor.org/packages/release/bioc/html/CAMERA.html                                                 | http://www.ncbi.nlm.nih.gov/pubmed/22111785                        | 21702951 |                                                                         |                                             |          |
| CCPN Metabolomics     | This is a metabolomics add on to the CcpNmr software. It contains fulled assigned 1D and 2D spectra of 76 of metabolites (those that are most commonly found in biofluids). It also contains predicted 1H-1H TOCSY spectra, that are not available on public repositories. It allows for the creation of custom, user defined libraries of spectra.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | NMR                            | 1D NMR2D NMR                      |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | Python                                            | 1.3                    | numpy, Tcl/Tk, Tix, Mesa                                                                                                                           | NMRPipe                                                                                          | Bruker, Varian                                                                                                             | 2011             | 2011               |                                                                                                                                                            | http://www.ccpn.ac.uk/collaborations/metabolomics                                                              | https://www.ncbi.nlm.nih.gov/pubmed/21216771                       | 25005748 |                                                                         |                                             |          |
| CFM-ID                | Competitive Fragment Modeling for Metabolite Identification (CFM-ID) is a web server designed for ESI-MS/MS data analysis. It uses single energy - competitive fragmentation modeling (SE-CFM) to predict MS/MS spectra, for MS/MS peak annotation and for putative metabolite annotation. There are two SE-CFM which were trained on the METLIN database, one for positive and one for negative modes. Spectra can be predicted for three collision energies: 10V, 20V and 40V, requiring an input of SMILES or InChI formats of chemicals of interest. Putative MS/MS peak annotation requires input of a peak table and a list of metabolites in SMILES or InChI format. Putative Metabolite Annotation can be performed using HMDB, KEGG or a user defined list of metabolites.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 2a - Library Spectrum Match                                                                                                                                                                                                                                                              | LC-MS                          | LC-MS/MS                          |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Ruby on the Rails, Javascript, MySQL, Redis, HTML | 2.4                    | -                                                                                                                                                  | Peak list, Metabolite list in SMILES/InChI format for annotation                                 | -                                                                                                                          | 2014             | 2015               | GPL-2.1                                                                                                                                                    | http://cfmid.wishartlab.com/                                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/24895432                        | 21414990 |                                                                         |                                             |          |
| COLMAR                | Complex mixture analysis by NMR (COLMAR) is designed for the analysis of covariance-NMR (2D) data of complex mixtures. There are three web-servers: one to calculate the NMR covariance spectrum from the raw input, one to decompose 2D covariance TOCSY spectrum into a reduced set of non-redundant 1D cross sections and one to screen the traces against a spectral database for metabolite identification via the BMRB and HMDB. Prior to processing there is the option to remove the water line.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | NMR                            | Covariance NMR                    |                                         | Basic            | Web Based                             | Web User Interface                               | Linux, OS X, Windows (Web-based)                      | -                                                 | 1                      | -                                                                                                                                                  | NMRPipe                                                                                          | Bruker, Varian                                                                                                             | 2011             | 2011               |                                                                                                                                                            | http://spin.ccic.ohio-state.edu/index.php/colmar                                                               | http://www.ncbi.nlm.nih.gov/pubmed/19634130                        | 19291281 |                                                                         |                                             |          |
| FingerID              | FingerID can be used for metabolite identification of MS/MS data using kernel methods. It used SVM to first predict a large set of molecular properties for matching against large databases (e.g. PubChem, MassBank, METLIN) for metabolite identification. This is an alternative technique to the traditional method of directly mapping the metabolite to the spectrum. Two types of kernel are used: integral mass accuracy kernel and high mass accuracy kernel. The molecular properties used are: topology, physio-chemical and electrical. A probabilistic score from the predicted molecular fingerprints is used to retrieve candidate molecules from metabolite databases.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 2a - Library Spectrum Match                                                                                                                                                                                                                                                              | LC-MS                          |                                   |                                         | Advanced         | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | MATLAB, Python                                    | 1.4                    | Python ł 2.7, Numpy ł 1.4.0, LibSVM 3.17 python interface                                                                                          | Peak list with m/z and RT                                                                        | -                                                                                                                          | 2012             | 2016               | Apache 2.0                                                                                                                                                 | https://github.com/icdishb/fingerid                                                                            | http://www.ncbi.nlm.nih.gov/pubmed/22815355                        | 27746707 |                                                                         |                                             |          |
| FragPred              | Multiple reaction modelling (MRM) is a QQQ mode. It has traditionally just been used for small data sets of authenticated standards. FragPred aims to extend MRM for all of the metabolites in the METLIN database. Logistic regression fitted with L1-regularized objective was used to model the likelihood of a molecule ionising and producing a structure in METLIN. This results in a list of predicted fragments.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          |                                   | Targeted                                | Advanced         | R Scripts                             | Command line interface                           | Linux, OS X, Windows                                  | R                                                 | 1                      | R                                                                                                                                                  | -                                                                                                | -                                                                                                                          | 2015             | 2015               |                                                                                                                                                            | http://pattilab.wustl.edu/software/FragPred/index.php                                                          | http://www.ncbi.nlm.nih.gov/pubmed/25691443                        | 22635605 |                                                                         |                                             |          |
| HR3                   | HR3 is designed for the annotation of elemental composition of metabolomics LC-MS data. It can be run either through a GUI or in batch mode. It contains a set of rules for filtering PubChem by elemental composition ratios to reduce the number of potential compound annotations.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | LC-MS                          |                                   |                                         | Advanced, Medium | Package                               | Command line interface, Graphical User Interface | Windows                                               | Visual C++                                        | -                      | -                                                                                                                                                  | Peak list                                                                                        | -                                                                                                                          | 2014             | 2014               | GPL-2                                                                                                                                                      | http://www.wur.nl/en/show/MetAlign.htm                                                                         | http://www.ncbi.nlm.nih.gov/pubmed/24820703                        | 26017271 |                                                                         |                                             |          |
| Lipid-Pro             | This provides metabolite identification for preprocessed DIA LC-MS/MS lipidomics data, using rt-aligned monoisotopic masses of molecular (MS) and fragment (MS/MS) ions. It is a modular based software, consisting of 5 modules: building block, precursor candidates, fragment candidates, mass analyzer and lipid identification. Firstly monoisotopic masses are calculated from elemental composition of the lipid building blocks (heads and tails). Different heads and tails are combined in the precursor candidates modules and potential adducts are identified. The monoisotopic masses of the expected fragments are then calculated. Putative fragments and molecular ions are then matched to the inputted m/z values. Lipid candidates are then identified after time-alignment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 2a - Library Spectrum Match                                                                                                                                                                                                                                                              | LC-MS                          | LC-MS/MS                          | Lipidomics                              | Advanced         | Package                               | Graphical user interface                         | Windows                                               | ?                                                 | 1                      | -                                                                                                                                                  | csv Peaklist                                                                                     | -                                                                                                                          | 2014             | 2015               | Academic users only                                                                                                                                        | http://www.neurogenetics.biozentrum.uni-wuerzburg.de/verbundprojekte/services/lipidpro/                        | http://www.ncbi.nlm.nih.gov/pubmed/25433698                        | 25859612 |                                                                         |                                             |          |
| MAGMa                 | MS Annotation based on in silico Generated Metabolites (MAGMa) automatically annotates MS^n data, where n ł 2, without requiring reference spectral data. Substructures are derived from in silico hierarchical trees. Assignments at each MS level take both the previous as well as the successive fragments into account. Candidate structures are ranked by scores. MAGMa automatically searches PubChem and HMDB.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 2a - Library Spectrum Match                                                                                                                                                                                                                                                              | LC-MS                          | LC-MS/MSLC-MS^n                   |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Python                                            | 1.0.0                  | -                                                                                                                                                  | mzXML, Tree Format                                                                               | -                                                                                                                          | 2013             | 2016               | Apache License 2.0                                                                                                                                         | http://www.emetabolomics.org/magma                                                                             | http://www.ncbi.nlm.nih.gov/pubmed/23662787                        | 22111785 |                                                                         |                                             |          |
| MetaboHunter          | This is a web-based server designed for automatic metabolite assignment of 1H NMR spectra. It includes three methods for metabolite identification: a novel scoring function, an iterative greedy selective approach designed to minimise the number of false positives whilst slightly increasing the number of false negatives and selection approaches with a user defined chemical drift to allow for the identification of metabolites from spectra obtained under slightly different conditions to the reference spectra. It also includes two reference libraries: 876 spectra from HMDB and 448 spectra from MMCD, that were manually curated. Spectra are first filtered for noise by setting a minimum intensity threshold. A metabolite is considered to be present if at least one of its peaks matches a reference peak at a significance score greater than a user defined threshold. The significance score is the total number of matched peaks/ total number of peaks + 1. It is designed to not favour metabolites with a greater number of peaks.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Basic            | Web Based                             | Web User Interface                               | Linux, OS X, Windows (Web-based)                      | PHP, Javascript, Perl                             | 1                      | -                                                                                                                                                  | List of Peaks with amplitutes, .txt                                                              | -                                                                                                                          | 2009             | 2015               |                                                                                                                                                            | http://www.nrcbioinformatics.ca/metabohunter/                                                                  | http://www.ncbi.nlm.nih.gov/pubmed/21999117                        | 21216771 |                                                                         |                                             |          |
| Metabolome Searcher   | Retrieves a list of metabolite identifications, based on chemical formula, SMILES structure and monoisotopic mass. This is done on a by species basis, in order to putatively label metabolites. The species lists are from ProCy, an online implementation of Pathway Tools.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          |                                   | Untargeted                              | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Perl                                              | 1                      | -                                                                                                                                                  | Text file with list of m/z values                                                                | -                                                                                                                          | 2015             | 2015               |                                                                                                                                                            | http://procyc.westcent.usu.edu/cgi-bin/MetaboSearcher.cgi                                                      | http://www.ncbi.nlm.nih.gov/pubmed/25887958                        | 24895432 |                                                                         |                                             |          |
| MetaboMiner           | This performs semi-automated metabolite quantification of 2D TOCSY and HSQC spectra. It contains reference libraries split by biofluid: CSF, plasma and urine, as well as a biofluid library and a total library. There are 223 TOCSY spectra (from BMRB and MRMD) and 502 HSQC spectra (from HMDB). A number of peak processing methods are included: streak removal and symmetry editing. Streaks (usually residual solvent signals) are found and removed by finding groups of peaks at common locations and removing them. Symmetrical editing is for TOCSY spectra, as TOCSY peak signals form a symmetrical square pattern along the diagonal line, and thus off diagonal peaks without symmetry can be considered as artifacts. A metabolite is identified if it satisfies the minimum signature requirement - the minimum peak set that can uniquely identify a compound from all others in the library. The false positive rate is reduced by having a minimum number of matched peaks and a minimum fraction of peaks. There are two compound identification techniques: minimal signature method (MS) and percentage match method (PM).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | NMR                            | 2D TOCSY NMR, 2D HSQC NMR         |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | Java                                              | 1                      | ł JRE 5.0                                                                                                                                          | List of peaks with optional amplitutes                                                           | -                                                                                                                          | 2008             | 2008               |                                                                                                                                                            | http://wishart.biology.ualberta.ca/metabominer/                                                                | http://www.ncbi.nlm.nih.gov/pubmed/19040747                        | -        |                                                                         |                                             |          |
| MetaboSearch          | Searches HMDB, MMCD, Metlin and LipidMaps for metabolite identification from a given list of m/z values, using batch search. CAMERA output (with adduct and ion annotation) can also be used as input. For input, data can be either directly pasted in or imported in the .xls format.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          |                                   | Untargeted                              | Basic            | Package                               | Graphical user interface                         | Unix/Linux, Mac OS, Windows                           | Java, C, MATLAB                                   | 1                      | JRE6/JRE7                                                                                                                                          | List of m/z values, List of m/z values with CAMERA annotation                                    | MarkerLynx                                                                                                                 | 2012             | 2012               |                                                                                                                                                            | http://omics.georgetown.edu/metabosearch.html                                                                  | http://www.ncbi.nlm.nih.gov/pubmed/22768229                        | 19634130 |                                                                         |                                             |          |
| MetFamily             | MetFamily provides preprocessing and metabolite identification using a novel combination of PCA and HCA. Initially algorithms from MS-DIAL are used for feature detection, adduct annotation, feature alignment and compound spectra extraction. The fragment matrix is then assembled with noise removal, neutral loss addition, fragment intensity normalisation, alignment of all MS/MS spectra and isotope removal. To detect group-discriminating features using MS1 abundances, PCA is used (scaling can be pareto or auto). HCA is then used to identify metabolite families based on MS/MS patterns leading to a list of regulated metabolite families.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          | LC-MS/MS                          | Untargeted                              | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.0.3                  | -                                                                                                                                                  | MS/MS library                                                                                    | -                                                                                                                          | 2016             | 2016               | GPL-3                                                                                                                                                      | http://msbi.ipb-halle.de/MetFamily/                                                                            | https://www.ncbi.nlm.nih.gov/pubmed/27452369                       | -        |                                                                         |                                             |          |
| MetFrag               | MetFrag is available as a web app, command line tool and R package, enabling identification of MSn data fragments that are not found in existing databases. It queries PubChem, KEGG Compounds and ChemSpider to identify candidate molecules and then generates all possible topological fragments for the candidate compound. These fragments can then be matched to the measured peaks by creating a fragmentation tree, using an iterative, breadth-first algorithm. It has recently been updated to allow users to search a wider selection of databases to identify candidate molecules to generate tological fragments from. Users can also select filtering criteria by inclusion or exclusion of substructures and elements.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | MSn                            |                                   |                                         | Basic            | Web App/ Package                      | Web user interface, Command line interface       | Unix/Linux, Mac OS, Windows                           | Java, R                                           | 2.2                    | R (ł 3.0.1) KEGGREST, rcdk 3.2.8.3                                                                                                                 | Peaklist                                                                                         | -                                                                                                                          | 2011             | 2015               | GPL (ł 2.1)                                                                                                                                                | http://c-ruttkies.github.io/MetFrag/                                                                           | https://www.ncbi.nlm.nih.gov/pubmed/26834843                       | 25160088 |                                                                         |                                             |          |
| MetFusion             | MetFusion combines MetFrag with MassBank. It aims to confirm the predicted result using spectral reference data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | MSn                            |                                   |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Java                                              | 1                      | -                                                                                                                                                  | Peaklist                                                                                         | -                                                                                                                          | 2012             | 2014               | GPL-3                                                                                                                                                      | http://msbi.ipb-halle.de/msbi/MetFusion                                                                        | https://www.ncbi.nlm.nih.gov/pubmed/23494783                       | -        |                                                                         |                                             |          |
| MetiTree              | This is designed for the annotation of MS/MS data. It integrates the MEF tool to assign elemental composition and neutral losses to the fragments. Newly acquired MS/MS data can also be compared to MS/MS data already in the library. There is also a spectral viewer to visualise the MS/MS data. Inputted data can be processed several times with different parameters to discover the best parameters to use for processing. MSn data can be labelled with InChI identifiers.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | LC-MS                          |                                   |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Javascript                                        | 1                      | -                                                                                                                                                  | .mzXML                                                                                           | -                                                                                                                          | 2012             | 2012               | GPL-2                                                                                                                                                      | http://www.metitree.nl/                                                                                        | https://www.ncbi.nlm.nih.gov/pubmed/22851531                       | 23829391 |                                                                         |                                             |          |
| MI-PACK               | Metabolite identification package (MI-PACK) uses transformation mapping (TM), mapping an experimentally derived empirical formula difference of a peak pair to a known substrate-product empirical formula difference derived from KEGG, for metabolite identification. It assumes that all the metabolites in a pathway will be present in a sample. Mass error surface from instrumental error is also calculated.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 |                                |                                   |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | Python, R                                         | 0.9                    | Python 2.6.4, SQLite Database Manger                                                                                                               | Peaklist .txt                                                                                    | -                                                                                                                          | 2010             | 2015               | GPL-3                                                                                                                                                      | http://www.biosciences-labs.bham.ac.uk/viant/mipack/                                                           | http://www.sciencedirect.com/science/article/pii/S0169743910000596 | 22884039 |                                                                         |                                             |          |
| MolFind               | MolFind This is designed for the putative identification of molecular formula. Candidate compounds are automatically downloaded from PubChem, but candidate compound lists downloaded from KEGG, HMDB or generated by Molgen can also be used for metabolite identification. This is not designed specifically for metabolomics.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | LC-MS                          | HPLC-MS                           |                                         | Basic            | Package                               | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | Java                                              | 1.9                    | ł Java 1.6                                                                                                                                         | m/z list                                                                                         | -                                                                                                                          | 2013             | 2014               |                                                                                                                                                            | http://metabolomics.pharm.uconn.edu/Software.html                                                              | http://www.ncbi.nlm.nih.gov/pubmed/23039714                        | 25549083 |                                                                         |                                             |          |
| MS-LAMP               | Mass Spectrometry based Lipid(ome) Analyzer and Molecular Platform is designed for metabolite identification of ESI and MALDI data. A list of molecular masses can be queried against the MS-LAMP database (consisting of lipids from LIPID MAPS and the MtB LipidDtB) for lipid identification. Users can also add their own lipid database, if in the correct format for MS-LAMP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | MS                             |                                   | Lipidomics                              | Advanced         | Package                               | Graphical user interface                         | Unix/Linux, Windows                                   | Perl                                              | -                      | Tk                                                                                                                                                 | List of m/z                                                                                      | -                                                                                                                          | 2012             | 2012               |                                                                                                                                                            | http://ms-lamp.igib.res.in/                                                                                    | http://www.ncbi.nlm.nih.gov/pubmed/23584940                        | 23418185 |                                                                         |                                             |          |
| MS2Analyzer           | This Fiehn lab tool allows metabolite identification of LC-MS/MS spectra, by searching a database of substructures of MS/MS spectra. It is particularly useful for identifying novel compound species that are not yet in tandem spectral databases. The four types of mass spectral features which can be searched for are: precursor ions, product ions, neutral losses and m/z differences. When data is inputted into the software, first spectral simplification is performed, by removing all features below a user defined intensity threshold. As well as substructures automatically generated by MS2Analyzer, 147 neutral losses were manually curated from Google Scholar. Implemented as both a GUI and command line for both novice users and bioinformaticians.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 2a - Library Spectrum Match                                                                                                                                                                                                                                                              | LC-MS                          | LC-MS/MS                          | Untargeted                              | Medium           | Package                               | Graphical user interface, Command Line Interface | Unix/Linux, Mac OS, Windows                           | Java                                              | 2.1                    | ł JRE 7                                                                                                                                            | .MSP (NIST), .MGF (Mascot generic format)                                                        | -                                                                                                                          | 2013             | 2014               | MIT                                                                                                                                                        | http://fiehnlab.ucdavis.edu/projects/MS2Analyzer/                                                              | http://www.ncbi.nlm.nih.gov/pubmed/25263576                        | 27584001 |                                                                         |                                             |          |
| MSFINDER              | This RIKEN software is designed for the putative identification of unknown metabolites in LC-MS/MS data, using a three stage process. Firstly, MS1 and MS/MS molecular formulas are predicted using the seven golden rules and a neutral loss database. The chemistry development kit (CDK) is then used to fragment structural data from PubChem matching these molecular formula. Finally candidate compounds are ranked using a likelihood score. Candidates are filtered by whether or not they are present in an existing metabolite database.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          | LC-MS/MS                          | Untargeted                              | Basic            | Package                               | Graphical user interface                         | Windows                                               | C++                                               | 2.08                   | ł .NET Famework 4.0                                                                                                                                | ?                                                                                                | ?                                                                                                                          | 2015             | 2016               |                                                                                                                                                            | http://prime.psc.riken.jp/Metabolomics_Software/                                                               | http://www.ncbi.nlm.nih.gov/pubmed/27419259                        | 24176773 |                                                                         |                                             |          |
| MyCompoundID          | MyCompoundID provides an online resource for Putative Metabolite Annotation. The MyCompoundID.org (MCID)(Li et al. 2013) database encompasses 8,021 endogenous human metabolites from HMDB(Wishart et al. 2013) and 375,809 predicted metabolites from the Evidence-based Metabolome Library (EML). It includes an automated MS/MS search program(Huan, Tang, et al. 2015) that searches a spectral database created using in silico fragmentation prediction, as well as MS search program. Batch searches can be performed using a CSV of a peak list generated from LC-MS/MS spectral analysis. There are also a number of tools for the identification of specific chemical groups including DnsID(Huan, Wu, et al. 2015) for dansylate labelled metabolites, PEP(Tang et al. 2014) search for di/tripeptides and IsoMS for isotopic labelling studies.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 2a - Library Spectrum Match                                                                                                                                                                                                                                                              | MS                             |                                   |                                         |                  | Web Based                             | Web User Interface                               | Unix/Linux, Mac OS, Windows                           | Java, Javascript                                  | -                      |                                                                                                                                                    |                                                                                                  |                                                                                                                            | 2013             | 2015               |                                                                                                                                                            | http://mycompoundid.org/mycompoundid_IsoMS/                                                                    | http://www.ncbi.nlm.nih.gov/pubmed/23373753                        | 22815355 |                                                                         |                                             |          |
| MZedDB                | MZedDB is a database of potential adducts and neutral losses, that can be searched by sum formula, accurate mass, nominal mass or metabolite name. It can thus provide potential annotations for accurate mass, including the ability to predict the empirical formula, whilst taking into account adducts, neutral losses and isotopes. For database searches: KEGG, Metacyc and HMDB can be searched. The user can specify searches for metabolites found in a specific organism by interfacing with KEGG. MZedDB can be interfaced with R using a supplied script.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | LC-MS                          | LC-MS (ESI)                       | Untargeted                              | Basic            | Web App/ R script                     | Web user interface                               | Unix/Linux, Mac OS, Windows                           | MySQL, Perl, PHP, R                               | 1                      | -                                                                                                                                                  | m/z values                                                                                       | -                                                                                                                          | 2008             | 2009               |                                                                                                                                                            | http://maltese.dbs.aber.ac.uk:8888/hrmet/index.html                                                            | http://www.ncbi.nlm.nih.gov/pubmed/19622150                        | -        |                                                                         |                                             |          |
| mzOS                  | It provides deisotoping, annotation of fragments and adducts, metabolite identification with LipidMaps and HMDB and annotation confidence estimation. It is designed to receive a xcms peak table as input, which must contain a single polarity only. The confidence estimation of annotation can be performed using either bayesian inference (using network presence/ missing metabolites) or observed/theoretical isotope distribution pattern.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          |                                   | Untargeted                              | Advanced         | Package                               | Command line Interface                           | Unix/Linux, Mac OS, Windows                           | Python                                            | 0.1.3                  | numpy, scipy, scikit-learn, pandas                                                                                                                 | Peaklist .txt/ .csv                                                                              | -                                                                                                                          | 2015             | 2016               | MIT                                                                                                                                                        | https://github.com/jerkos/mzOS                                                                                 |                                                                    | 25691443 |                                                                         |                                             |          |
| ProbMetab             | This package performs compound assignment on already peak grouped LC-MS data using bayesian modeling. It suggests using a combination of XCMS, CAMERA and mzMatch for data preprocessing. It provides the functionality to combine molecular ion by CAMERA annotation between acquision modes. A non-redundant set of molecular ion annotations can then be extracted. Included in the package is a database of compounds to reactions from KEGG (there is also the option a user importing their own compound to reaction database). If an assement of carbon offset is available this can be used to improve predictions. The likelihood of each mass to compound assignment is then calculated, using carbon isotope ratio information if available. The reactions between the identified metabolites are then codified. Analysis flux can also be incorporated at this stage. A reaction matrix can then be created from the annotated molecular ions and the database. The ratio between observed and theoretical isotope patterns is then calculated (if carbon labelling is available). The likelihood of each mass to compound assignment is then calculated using mass accuracy and isotopic pattern if available. The posterior probabilities are then calculated according to the model (using the Gibbs sampler). At this stage a table of identified metabolites can be exported as an R matrix. Identified metabolites are then mapped to pathways that can be visualised in cytoscape.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1                      | CAMERA, graph, RCytoscape, multtest, RcppAramadillo, hwriter, GeneNet, RCurl, XML, rjson                                                           | mzData, mzML, mzXML, netCDF                                                                      |                                                                                                                            | 2013             | 2013               | GPL                                                                                                                                                        | http://labpib.fmrp.usp.br/methods/probmetab/                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/24443383                        | 26913198 |                                                                         |                                             |          |
| PUTMEDID-LCMS         | This is a Taverna based tool that provides modules which integrate into a workflow for analysing LC-MS data, including annotation and putative metabolite annotation. The three modules are for metabolite feature annotation, metabolite annotation and correlation analysis. Firstly the (already peak picked?) data undergo correlational analysis using either Pearson or Spearman rank correlations in the List_CorrData module. Next features are annotated as isotopes, adducts, dimers or others using accurate m/z difference and RT and median peak area in the annotate_MassMatch module. In the final module, matchMMF_MF, metabolites are putatively identified using the Manchester Metabolomics Database, which is built from HMDB, KEGG, LipidMaps, BioCyc and DrugBank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          | UPLC-MS                           |                                         | Advanced         | Framework, Workflow management system | Command line interface                           | Unix/Linux, Mac OS, Windows                           | Beanshell (Java)                                  | 1                      | Taverna                                                                                                                                            | Peaklist .txt                                                                                    | -                                                                                                                          | 2011             | 2011               |                                                                                                                                                            | http://www.mcisb.org/resources/putmedid.html                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/21325300                        | -        |                                                                         |                                             |          |
| Rdisop                | This bioconductor package is designed to determine the sum formula of metabolites based on their exact mass and isotopic pattern from high resolution MS. It integrates with XCMS, MassSpecWavelet and rpubchem. Input is a list of masses with normalised abundances that correspond to the isotope pattern of the same molecule. Firstly all elemental compositions that share a property of the target m/z are calculated, then compositions which do not exist in nature are removed and binding rules are applied. Unsaturated compositions are then removed. The theoretical isotope distribution pattern is then calculated and compared to the measured distribution pattern.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | MS                             | FT-ICR-MS, QTOF-MS                |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.34.0                 | R (ł 2.0.0), RcppClassic                                                                                                                           | Peaklist                                                                                         | -                                                                                                                          | 2008             | 2016               | GPL-2                                                                                                                                                      | http://bioconductor.org/packages/release/bioc/html/Rdisop.html                                                 |                                                                    | 25350247 |                                                                         |                                             |          |
| SIRIUS                | Sum formula identification by ranking isotope patterns using mass spectrometry (SIRUS) resolves the sum formula of a compound from its fragmented features using PubChem. The sum formula is calculated using the input mass and the natural isotope distribution. It does not aim to identify the 2D structure of the compound - only the sum formula. As of version 3.2 SIRIUS now has CSI:FingerID integration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 4 - Unequivocal Molecular Formula                                                                                                                                                                                                                                                        | MS                             | MSn                               |                                         | Advanced         | Package                               | Command line interface, Graphical user interface | Unix/Linux, Mac OS, Windows                           | Java                                              | 3.2                    | -                                                                                                                                                  | Normalised Peaklist (.csv)                                                                       | -                                                                                                                          | 2009             | 2016               | LGPL                                                                                                                                                       | https://bio.informatik.uni-jena.de/software/sirius/                                                            | http://www.ncbi.nlm.nih.gov/pubmed/19015140                        | -        |                                                                         |                                             |          |
| SpinAssign            | SpinAssign is a statistical index designed for the simultaneous annotation of many metabolites. It is a web-based application. Queried peaks are matched against a database of 1H and 13C chemical shifts and the p-values are calculated, where 1 is a 100% match. The database contains more than 1700 13C HSQC peaks, corresponding to 270 metabolites. The uniqueness (the extent to which a peak overlaps the reference chemical peak) is calculated. The annotation results are linked to KEGG.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Annotation                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | NMR                            | 2D 1H, 13C-HSQC NMR 1D NMR        |                                         | Basic            | Web Based                             | Web User Interface                               | Unix/Linux, Mac OS, Windows (Web-based)               | Javascript, HTML, PHP, MySQL                      | 1                      | -                                                                                                                                                  | List of peaks                                                                                    | -                                                                                                                          | 2009             | 2015               |                                                                                                                                                            | http://prime.psc.riken.jp/?action=nmr_search                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/20128615                        | 24820703 |                                                                         |                                             |          |
| VaLID                 | Visualization and Phospholipid Identification (VaLID) enables the prediction of the identities of lipid unknowns that are not present in the LipidMaps database. It contains exact and average masses of every theoretically possible phosphocholine, phosphoserine, phosphoethanolamine, glycerophosphoglycerol, glycerophosphoglycerolphosphate and cytidine 50-diphosphate 1,2-diacyl-sn-glycerol. It takes into account user-specified MS conditions.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Annotation                                                                                                                                                                                                                                                                                                                                          | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | MS                             |                                   | Lipidomics                              | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Java                                              | 3.2.0                  | Java                                                                                                                                               | List of m/z (exact/ average)                                                                     | -                                                                                                                          | 2013             | 2015               |                                                                                                                                                            | http://neurolipidomics.ca/                                                                                     | http://www.ncbi.nlm.nih.gov/pubmed/23162086                        | 26382193 |                                                                         |                                             |          |
| MetaQuant             | This is designed for the quantification of GC-MS data and also provides metabolite identification. A mixture of internal standards with known concentrations must first be run, allowing MetaQuant to solve the integrals of the peaks for calibration. The known concentrations are then used, along with peak areas as calibration points for regression analysis. There are a number of different regression methods available: linear, polynomial, power or exponential regression. For quantification, peaks of interest are first identified and then integrated and the appropriate regression function is used.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Annotation, Other Tools                                                                                                                                                                                                                                                                                                                             |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   | Targeted                                | Basic            | Web App                               | Web user interface                               | Linux, Windows (Precompiled), OS X                    | Java                                              | 1.3                    | JRE                                                                                                                                                | netCDF, CSV                                                                                      | -                                                                                                                          | 2006             | 2008               | GPL                                                                                                                                                        | http://bioinformatics.org/metaquant/                                                                           | http://www.ncbi.nlm.nih.gov/pubmed/17046977                        | 22308147 |                                                                         |                                             |          |
| Credential Features   | This package is designed to optimise LC-MS workflows where 1000s of features are detected from a single run. It uses credentialling to identify features which are artificial as opposed to those of biological origin. To do this E.coli were grown on either natural abundance media or uniformly C13 (UC13) media. These two media were then mixed at different ratios (1:1 and 1:2). Once LC-MS analysis has then be performed. The credential features package is used for data analysis. XCMS and CAMERA are utilized for preprocessing and annotation respectively. The credential features algorithm performs two rounds of filtering to identify biological features. Firstly, coeluting peaks within a single sample are assessed for potential isotopologue pairs and the ratios of 12C:13C are then evaluated and those that are not within a set percentage are discarded. The samples with different ratios of C12:C13 are then taken together for a second round of filtering.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Other Tools                                                                                                                                                                                                                                                                                                                                         | Experimental Optimisation                                                                                                                                                                                                                                                                      | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1                      | xcms, data.table, plyr, CAMERA, ggplot2, gridExtra, Hmisc                                                                                          |                                                                                                  | -                                                                                                                          | 2014             | 2015               |                                                                                                                                                            | http://pattilab.wustl.edu/software/credential/credential.php                                                   | http://www.ncbi.nlm.nih.gov/pubmed/25160088                        | 26784691 |                                                                         |                                             |          |
| ICT                   | Isotope correction toolbox (ICT) can be used for isotope correction of MS/MS data. It can be applied to any labeling source and any element and isotope can be considered. When the input is read, the correction matrix is generated. First the ratio of isotopes in the precursor and product ions are calculated and then combined as ratios and the ratios of individual isotopes are combined. The probability of occurrence of each isotope ratio is calculated and these are used to create the correction matrix. Negative correction ratios can arise and these are set to zero.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                | MSMS/MS                        |                                   |                                         | Advanced         | Toolkit                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | Perl                                              | 0.04                   | Perl                                                                                                                                               | List of metabolites .txt                                                                         | -                                                                                                                          | 2015             | 2015               | Artistic 2.0                                                                                                                                               | https://github.com/jungreuc/isotope_correction_toolbox/                                                        | http://www.ncbi.nlm.nih.gov/pubmed/26382193                        | 24990606 |                                                                         |                                             |          |
| iontree               | Ion fragmentation data analysis, for either LC-MS or DI-MS. Ion trees are constructed for MSn data. A relational database can be constructed for management of the ion tree data. Distance matrices for MS2 data can be constructed to measure spectral similarity.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                | LC-MSDI-MS                     | LC-MS/MSDI-MS/MS                  |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.20.0                 | methods, rJava, RSQLite, XML                                                                                                                       | mzXML                                                                                            | -                                                                                                                          | 2012             | 2016               | GPL-2                                                                                                                                                      | http://www.bioconductor.org/packages/release/bioc/html/iontree.html                                            | -                                                                  | -        |                                                                         |                                             |          |
| IosCor                | This was developed for isotopic correction of all isotopes, including rare ones e.g. 57Fe and elements with more than two isotopes e.g. O (16,17 and 18), using a matrix-based method. It can be applied to both natural abundance isotopic variance and isotopically labeled metabolites. To avoid biases the optimisation method includes constraints on the isotopic fraction values. It can only handle MS and not MS/MS data. It is not possible for negative correction values to be produced.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                | MS                             |                                   | Targeted                                | Advanced         | Python Package                        | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | Python                                            | 1                      | Python 2.6+ (< 3.0), NumPy (1.6.0.2), SciPy (0.9.0.1)                                                                                              | List of metabolite and elemental formulas                                                        | -                                                                                                                          | 2011             | 2011               | EDL-2                                                                                                                                                      | http://metasys.insa-toulouse.fr/software/isocor/                                                               | http://www.ncbi.nlm.nih.gov/pubmed/22419781                        | 24004415 |                                                                         |                                             |          |
| IPO                   | Isotopologue parameter optimisation (IPO) is designed to increase the reliability of peak picking (centWave and matchedFilter), retention time alignment (orbiwarp) and grouping (density) of XCMS, by optimising the parameters. A design of experiments approach (DoE) is used to optimised parameters. Box-Behnken designs (BBD) are used to create DoE. Feature detection parameters are optimised first, followed by simultaneous optimisation of retention time correction and group parameters. Peak picking scores are calculated based on the ratio of reliable peaks (isotopologues). Relative retention time deviations are minimised to produce the retention time correction score (RCS) using the inverse of the average retention time score. The ratio of the squared reliable groups to non-reliable groups is the grouping scores (GS). The norms of the RSC and GS are then combined as the retention time correction and grouping target value (RGTV). These scores are the evaluated using response-surface models. The centre of the next DoE is the combination of parameters that produced the best score in the response-surface models.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Other Tools                                                                                                                                                                                                                                                                                                                                         | Software Optimisation                                                                                                                                                                                                                                                                          | LC-MS                          |                                   | Untargeted                              | Medium           | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.7.5                  | CAMERA, xcms, rsm, devtools                                                                                                                        | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2015             | 2016               | GPL-2                                                                                                                                                      | https://github.com/glibiseller/IPO                                                                             | http://www.ncbi.nlm.nih.gov/pubmed/25888443                        | 22419781 |                                                                         |                                             |          |
| MetaboQC              | MetaboQC provides a post-peaking QC procedure. The user must first run the peak picking software of their choice, under random parameters for 50-200 iterations. Each peak picking output (PP-output) then must be normalised inside each replicate group. The inter replicate correlation (AvCorr) and z-correlation (Zcorr) is then estimated across replicate groups. The PP-output with the best Zcorr and high enough AvCorr is then selected. There is also discrimination between replicate groups with high Zcorr and AvCorr and those with low to find parameters that provide high Zcorr and Avcorr. The peak picking software should then be run with these parameters. MetaboQC can then be used to detect local discrepancies between pairs of replicates and removing sample outliers. The output of MetaboQC then provides the parameter conditions with the least defects. The user can then manually remove features that are enriched in low score replicates                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Other Tools                                                                                                                                                                                                                                                                                                                                         | Software Optimisation                                                                                                                                                                                                                                                                          | LC-MS                          |                                   | Untargeted                              | Advanced         | Package                               | Command line interface                           | Windows (32 bit)                                      | C++                                               | 1                      | -                                                                                                                                                  | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2010             | 2013               |                                                                                                                                                            | http://evolution.haifa.ac.il/index.php/component/k2/item/146                                                   | http://www.ncbi.nlm.nih.gov/pubmed/20977194                        | 25888443 |                                                                         |                                             |          |
| MetShot               | MetShot provides nearline data-dependent acquisition (DDA) LC-MS/MS processing strategies. With its pipeline, it is designed for the optimum collection of LC-MS/MS data. Firstly LC-MS raw data must be gathered and analysed (preprocessing with XCMS and CAMERA) to identify features of interest. These features of interest can then be analysed with DDA LC-MS/MS. It generates target peak lists for MS/MS analysis from single LC-MS runs (for Bruker micrOTOF-Q, Agilent QTOF and Waters QTOF). Once LC-MS/MS data has been collected, peaks are identified using the centWave algorithm and grouped using CAMERA algorithms.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Other Tools                                                                                                                                                                                                                                                                                                                                         | Experimental Optimisation                                                                                                                                                                                                                                                                      | LC-MS/MS                       |                                   | Targeted                                | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.3.2                  | CAMERA, xcms, XML, class, Biobase                                                                                                                  | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2012             | 2016               | GPL (ł 2)                                                                                                                                                  | https://github.com/sneumann/MetShot                                                                            | http://link.springer.com/article/10.1007%2Fs11306-012-0401-0       | 25887233 |                                                                         |                                             |          |
| MetSizeR              | MetSizeR is a CRAN package that provides a GUI for estimating the sample size required for targeted and NMR metabolomics experiments, even when pilot studies are not available. When a pilot study is not available, pseudo-metabolomic data is simulated with a statistical model. The statistical model used depends on which of probabilistic principal components analysis (PPCA), probabilistic principal components and covariates analysis (PPCCA) or dynamic probabilistic principal components analysis (DPPCA) the user intends to use for analysis of the studies data. The user must specify the desired FDR and the optimal sample size is the one with this FDR. When there is a pilot study available the Test Statistic is estimated using a permutation technique, maintaining between subject variability and noise level.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Other Tools                                                                                                                                                                                                                                                                                                                                         | Experimental Design                                                                                                                                                                                                                                                                            | NMR MS                         |                                   | Targeted                                | Medium           | R Package                             | Graphical user interface                         | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.1                    | mvtnorm, MetabolAnalyze, cairoDevice, gWidgets, gWidgetsRGtk2                                                                                      | .txt/ none                                                                                       | -                                                                                                                          | 2014             | 2015               | GPL-2 | GPL-3                                                                                                                                              | https://cran.r-project.org/web/packages/MetSizeR/index.html                                                    | http://www.ncbi.nlm.nih.gov/pubmed/24261687                        | 21169379 |                                                                         |                                             |          |
| MSClust               | Software designed to reduce mass signal redundancy of untargeted metabolomics data using mass signal correction analysis. First the number of clusters is determined in an unsupervised manner using the potential density method, then subtractive fuzzy clustering is used to cluster peaks around the centre (centrotype). Features are clustered based on similarity of chromatography (retention time) and similarity of ion fragment patterns. A list of peaks is then generated.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   |                                         | Advanced         | Package                               | Command line interface                           | Windows                                               | C++                                               | 1                      | -                                                                                                                                                  | .txt peak list (no zero values)                                                                  | -                                                                                                                          | 2011             | 2012               |                                                                                                                                                            | http://www.wageningenur.nl/nl/Expertises-Dienstverlening/Onderzoeksinstituten/rikilt/show/MetAlign.htm         | http://www.ncbi.nlm.nih.gov/pubmed/22833709                        | 17165823 |                                                                         |                                             |          |
| msPurity              | msPurity assess the purity of MS/MS precursors for both LC-MS/MS and DI-MS/MS data, describing the contribution of the selected precursor using the 'precursor purity' metric. The Michalski approach (intensity of selected precursor divided by total intensity of the isolation window) with the exception that the metric is interpolated at the time of the MS/MS scan, is used to calculate precursor purity. It can be interfaced with an xcmsSet object to filter out spectra that are below a user-specified threshold of purity. It can also perform background subtraction using peaks detected in blank samples.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                | LC-MSDI-MS                     | LC-MS/MSDI-MS/MS                  |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.1.0                  | Rcpp                                                                                                                                               | .mzML, .csv                                                                                      | -                                                                                                                          | 2016             | 2016               | GPL (>= 2)                                                                                                                                                 | https://bioconductor.org/packages/devel/bioc/html/msPurity.html                                                | -                                                                  | 25433698 |                                                                         |                                             |          |
| mzR                   | This bioconductor package provides a common interface for several mass spectrometry data formats: .mzML, .mzXML, .mzData, netCDF and .mzIdentML. It acts entirely as a parser and does not perform any processing, which can instead be performed with other packages.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                | MS                             |                                   |                                         | Advanced         | R Package                             | Command line Interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 2.8.1                  | Rcpp (ł 0.10.1), methods, utils                                                                                                                    |                                                                                                  |                                                                                                                            | 2011             | 2016               | Artistic-2.0                                                                                                                                               | http://bioconductor.org/packages/release/bioc/html/mzR.html                                                    | http://www.ncbi.nlm.nih.gov/pubmed/15529173                        | 17720531 |                                                                         |                                             |          |
| QCScreen              | This tool allows for the quality assessment of HRLC-MS data. It provides visualisation of the stability and precision of the chromatographic separation step and the mass accuracy and m/z value over time. Based on sample names QCScreen automatically assigns them to categories e.g. blank, QC, matrix QC or biological sample. Target species must be user specified, either within the GUI or as a tab separated list. Theoretical m/z and rt values are compared to measured values to obtain the deviation. Results are coloured from green to red to mark whether a parameter is within the expected values or not. The relative isotopologue abundance of 13C:12C can be calculated and plotted. The results of QCScreen can be used as input parameters of other software e.g. XCMS. The raw, unprocessed data in .mzXML format can be used for analysis.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Other Tools                                                                                                                                                                                                                                                                                                                                         | Quality Assessment                                                                                                                                                                                                                                                                             | LC-MS                          | Centroid HRLC-MS                  |                                         | Medium           | Package                               | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | Python, R                                         | 1.3                    | R (ł 2.12.0), waveslim, signal, ptw, MassSpecWavelet                                                                                               | Peaklist .csv, .mzXML                                                                            | -                                                                                                                          | 2015             | 2016               | GPL-2                                                                                                                                                      | https://sourceforge.net/projects/qcscreen/                                                                     | http://www.ncbi.nlm.nih.gov/pubmed/26498454                        | 22272252 |                                                                         |                                             |          |
| RMassBank             | This bioconductor package is designed to allow for easy upload of data onto MassBank. Initially MS/MS spectra are extracted from raw data files. Then using a .csv containing the compound information, RMassBank attempts to fit molecular formulas for every peak. This generates an array of metabolites, which is then aggregated into a list. A calibration curve is then calculated from this list and all spectra are then recalibrated. The recalibrated spectra are then reanalysed and aggregated and know electronic noise is removed. Peaks without a compound identification are then reanalysed, allowing for N2O as an additional element. After this peaks that are only found in a single spectra are removed. After this annotated spectra can be uploaded onto MassBank. Additionally XCMS can be used to extract the MS2 spectra (optionally the MS1) and then for peak picking. CAMERA can then be used to process the spectra.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Other Tools                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                |                                |                                   | Targeted                                | Advanced         | R Package                             | Command line Interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 2.2.1                  | Rcpp                                                                                                                                               | mzML, mzData                                                                                     | -                                                                                                                          | 2013             | 2016               | Artistic-2.0                                                                                                                                               | https://bioconductor.org/packages/release/bioc/html/RMassBank.html                                             | http://www.ncbi.nlm.nih.gov/pubmed/23303751                        | 23662787 |                                                                         |                                             |          |
| batchCorr             | This is designed for between and within-batch correction of LC-MS data. There are functions for batch alignment, drift correction and batch normalisation. For batch alignment feature missingness is aggregated on a batch level, features with missingness that are sufficiently similar m/z and retention time and ensuring orthogonal batch presence among feature alignment candidates. Drift correction is performed using clustering (mclust) on features in observation space. It provides a trade-off between the power per drift pattern and the multiple drift patterns within the dataset. For batch normalisation either QC/Reference or population (median) normalisation can be used. A quality heuristic is used to determine whether the QC/Reference is suitable for normalisation; if it is not, median normalisation is used instead.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Post-processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.1.7                  | R (ł 2.10)                                                                                                                                         | Peaklist, xcmsSet                                                                                | -                                                                                                                          | 2016             | 2016               | GPL-2                                                                                                                                                      | https://gitlab.com/CarlBrunius/batchCorr                                                                       | http://www.ncbi.nlm.nih.gov/pubmed/27746707                        | 24642061 |                                                                         |                                             |          |
| crmn                  | Cross-contribution robust multiple standard normalization (crmn) is a CRAN package which provides normalisation for metabolomics data. crmn is a normalisation technique which can be used to normalise metabolomics data which has been generated with internal standards. A number of other normalisation methods are also included: NOMIS, One (divide each analyte by a single user-defined IS), RI (divide each analyte by the abundance estimate of the IS, closest in terms of RI), totL2 (ensures the square sum of each sample is the same), median and average normalisation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.0.20                 | R (ł 2.10), pcaMethods (ł 1.56.0), Biobase, methods                                                                                                | Matrix, ExpressionSet                                                                            | -                                                                                                                          | 2014             | 2015               | GPL (ł 3)                                                                                                                                                  | https://cran.r-project.org/web/packages/crmn/index.html                                                        | -                                                                  | 24965016 |                                                                         |                                             |          |
| EigenMS               | EigenMS provides normalisation of LC-MS metabolomics data using a singular value decomposition-based method. Firstly variation due to differences between experimental conditions is captured using ANOVA. Trends caused by bias are found by applying singular value decomposition to a matrix of residuals. These bias trends are then removed from the dataset.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-MS/MS                          |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R, MATLAB                                         | Mar-14                 | -                                                                                                                                                  | Peak list                                                                                        | -                                                                                                                          | 2009             | 2015               | GPL-2                                                                                                                                                      | https://sourceforge.net/projects/eigenms/                                                                      | http://www.ncbi.nlm.nih.gov/pubmed/25549083                        | 26279687 |                                                                         |                                             |          |
| KMDA                  | Kernel-Based Metabolite Differential Analysis (KMDA) is a CRAN package, which provides a kernel-based score test to cluster metabolites between treatment groups, in order to handle missing values. There are two kernels: one distance-based and the other stratified, in order to capture both the discrete and continuous nature of metabolomics data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                |                                |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1                      | R (ł 2.10)                                                                                                                                         | Peaklist (matrix)                                                                                |                                                                                                                            | 2014             | 2015               | GPL-2 | GPL-3                                                                                                                                              | https://cran.r-project.org/web/packages/KMDA/                                                                  | http://www.ncbi.nlm.nih.gov/pubmed/25887233                        | 25654076 |                                                                         |                                             |          |
| metabomxtr            | This bioconductor packages uses the Bernoulli/lognormal mixture-model (Moulton and Halsey, 1995) to formally account for individual metabolite 'missingness". It simultaneously estimates parameters                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | modeling the probability of a non-missing response and the mean of observed values. To do this it requires the input of the parameter of interest (i.e. the phenotypic difference) and the number of samples per group. The model can be controlled for confounding factors. After the full model containing the parameter of interest has been run | a reduced model without the parameter of interest is run. The significance of the full and reduced models is then compared using nested likelihood chi-squared tests. A results table can then be generated. This package is designed to be run for individual metabolites with missing values | not across the whole dataset." | Post-Processing                   |                                         | GC-MSLC-MS       |                                       | Untargeted                                       | Advanced                                              | R Package                                         | Command line interface | Unix/Linux, Mac OS, Windows                                                                                                                        | R                                                                                                | 1.8.0                                                                                                                      | methods, Biobase | Matrix, Data frame | -                                                                                                                                                          | 2014                                                                                                           | 2016                                                               | GPL-2    | https://www.bioconductor.org/packages/release/bioc/html/metabomxtr.html | http://www.ncbi.nlm.nih.gov/pubmed/25075114 | 21049934 |
| Metabonorm            | This is program for the normalisation of metabolite concentrations for low concentration metabolites. It uses a mixed model with simultaneous estimation from a correlation matrix. It is distributed as an R script.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | R                                                 | 1                      | gplots, hglm, affy, limma                                                                                                                          | Data frame                                                                                       | -                                                                                                                          | 2014             | 2014               | GPL-2                                                                                                                                                      | https://sourceforge.net/projects/metabnorm/                                                                    | http://www.ncbi.nlm.nih.gov/pubmed/24711654                        | 23918246 |                                                                         |                                             |          |
| MetabR                | This R package provides linear mixed-effect modeling (with regression and ANOVA terms) normalisation and statistical comparison between samples. Multiple different kinds of linear modeling are available: fixed linear models and linear mixed models. These models still require that an IS is present. The Shapiro-Wilk test is included to test for normality and Levene's test of equality of variance is also included. Pairwise mean fold changes are then calculated between treatment groups and heat maps of metabolites with significantly different signal intensities between treatment groups are visualised.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | R                                                 | -                      | -                                                                                                                                                  | csv, txt Peaklist                                                                                | -                                                                                                                          | 2012             | 2012               | GPL (ł 2)                                                                                                                                                  | http://metabr.r-forge.r-project.org/                                                                           | http://www.ncbi.nlm.nih.gov/pubmed/23111096                        | 24856452 |                                                                         |                                             |          |
| MetNorm               | This CRAN package provides RUV-random normalisation for metabolomics data and also RUV-random method for clustering. Remove unwanted variation-random (RUV-random) attempts to remove all unwanted variation: both biological and technical variation. RUV-random is suitable for both supervised and unsupervised methods. The RUV-random method for clustering can be used for unsupervised exploratory clustering purposes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | GC-MSLC-MSNMR                  |                                   |                                         | Advanced         | R Package                             | Command line interface/ Graphical user interface | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.1                    | -                                                                                                                                                  | Matrix                                                                                           | -                                                                                                                          | 2014             | 2015               | GPL-2 | GPL-3                                                                                                                                              | https://cran.r-project.org/web/packages/MetNorm/index.html                                                     | http://www.ncbi.nlm.nih.gov/pubmed/25692814                        | -        |                                                                         |                                             |          |
| MSPrep                | This is designed to provide functions that are not incorporated in preprocessing but are required for statistical analysis. It includes missing value imputation, filtering, normalisation and averaging of technical replications. It requires aligned LC-MS intensity data, a dataset linking subject ID to data and a clinical dataset. Firstly technical replicates are summed then averaged; CV is used a user specified threshold as to whether a feature is further included in the analysis. Data can then be filtered by a user-specified cutoff of a feature being required to be present in a certain percentage of samples to be included further. There are then three options for handling missing data: replace with zeros (assuming missing data are true zeroes), replace with one half of the minimum observed value for the compound(assuming missing compounds were below detectable limit) and Bayesian PCA (assuming the compound is present, but failed to be accurately detected). There are then five normalisation options: median, quantile, cross-contribution compensating multiple standard normalisation (CRMN), surrogate variable analysis (SVA) and removal of unwanted variation (RUV).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Post-Processing                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.1                    | pcaMethods (>= 1.24.0), crmn, preprocessCore, sva                                                                                                  | Peaklist                                                                                         | -                                                                                                                          | 2013             | 2014               | GPL-3                                                                                                                                                      | https://sourceforge.net/projects/msprep/                                                                       | http://www.ncbi.nlm.nih.gov/pubmed/24174567                        | 16808440 |                                                                         |                                             |          |
| metabolomics          | This CRAN package provides statistical analysis tools for metabolomics data. Functions provided include: HCA, Fold change analysis, heat maps, linear models (either ordinary statistics or empirical Bayes statistics), PCA and volcano plots. It also includes log transformation, missing value replacement and normalisation (median, mean, sum, ref, IS, normalization using optimal selection of multiple internal standards (normis), cross-contribution compensating multiple internal standard normalisation (ccmn) and remove unwanted variation (ruv2)). The ruv2 method of normalisation attempts to capture biological variation: both observed and unobserved. It generates a matrix of unwanted variation using non-changing metabolites, including internal and external standards.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Post-Processing, Statistical Analysis                                                                                                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                | MSNMR                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.1.4                  | limma, crmn, gplots, R (ł 3.1.0)                                                                                                                   | Data frame                                                                                       | -                                                                                                                          | 2014             | 2015               | GPL-2 | GPL-3                                                                                                                                              | https://cran.r-project.org/web/packages/metabolomics/index.html                                                | -                                                                  | 26247233 |                                                                         |                                             |          |
| muma                  | Metabolomics Univariate and Multivariate Analysis (muma - a CRAN package) provides a pipeline for analysis, including preprocessing, statistical analysis and metabolite identification. It also specifically provides a number of analysis tools for NMR data - statistical total correlation spectroscopy (STOCSY) and ratio analysis of NMR spectroscopy (RANSY). Initially a dataset is read and then screened for missing values. Four imputation methods are provided: mean, minimum, half-minimum and zero. Normalisation is then performed for the whole spectra, every variable is transformed into a fraction of the total intensity of the spectrum. A range of scaling techniques are then available: Auto, Pareto, Range, Vast and Medium, with each variable being mean centred before scaling. It provides a variety of both univariate (Welch's T test and Wilcoxon-Mann Whitney) and multivariate (PCA, PLS-DA and OPLS-DA) statistical techniques, as well a method of evaluating the clustering power of PCA. For univariate analysis a decision tree algorithm performs hypothesis tests on a single variable - first Shapiro Wilk's test is used to assess whether the data are normally distributed or not, then either Welch's T test or Wilcoxon-Mann Whitney test are performed. Benjamini-Hochberg multiple testing correction can then be applied. Correlation heatmaps can be created to visualise the data. With NMR data, these heatmaps are referred to as STOCSY and correlations >0.95 indicate a structural relationship between peaks belonging to the same molecule. RANSY is also designed to show the structural relationships between peaks.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Post-processing, Statistical Analysis                                                                                                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                | MSNMR                          |                                   |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | R                                                 | 1.4.0                  | car, pdist, pls, gplots, mvtnorm, robustbase, gtools, bitops, caTools, pcaPP, rrcov                                                                | .csv                                                                                             | -                                                                                                                          | 2012             | 2015               | GPL-2                                                                                                                                                      | https://cran.r-project.org/web/packages/muma/index.html                                                        | http://www.eurekaselect.com/107837/article                         | 21697128 |                                                                         |                                             |          |
| ALLocator             | This is designed for the analysis of LC-ESI-MS data, including data which has been 13C labelled. It includes both pre-processing, interpretation and metabolite identification. Peaks are detected using XCMS, spectra can then be deconvoluted using either CAMERA or ALLocatorSD and then KEGG Compounds for putative identification and pseudo fragment spectra can be matched against MassBank. ALLocatorSD is able to process 13C labelled data, whereas CAMERA is not. Also CAMERA and ALLocatorSD used different predefined lists of adducts and neutral losses.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                |                                | LC-ESI-MS                         |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Java, R, JavaScript                               | 1                      | -                                                                                                                                                  | mzData, mzXML, netCDF                                                                            | -                                                                                                                          | 2014             | 2014               |                                                                                                                                                            | https://allocator.cebitec.uni-bielefeld.de/landing.htm                                                         | http://www.ncbi.nlm.nih.gov/pubmed/25426929                        | 25897128 |                                                                         |                                             |          |
| alsace                | This bioconductor package provides tools for retention time alignment and intensity drift correction of spectra that are measured over time. It is an add on to the ALS package which implements alternating least squares (multivariate curve resolution) for analysis of chemical data, in particular hyphenated data where the first direction is a retention time axis, and the second a spectral axis e.g. HPLC-DAD (high-performance liquid chromatography with photodiode array detection). It also provides filtering of peaks below an intensity threshold.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | DAD                            | HPLC-DAD                          |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.10.0                 | R (ł 2.10), ALS, ptw (ł 1.0.6)                                                                                                                     | R accessible format (.csv)                                                                       | -                                                                                                                          | 2014             | 2016               | GPL (ł 2)                                                                                                                                                  | https://www.bioconductor.org/packages/release/bioc/html/alsace.html                                            | -                                                                  | 21999117 |                                                                         |                                             |          |
| AMDIS                 | Automated Mass Spectral Deconvolution and Identification System (AMDIS) provides deconvolution and identification of GC-MS data. It was originally designed for the automatic identification of chemical weapons, but is applicable to all GC-MS data. Firstly, GC-MS data are deconvoluted to extract pure compound peaks free of overlapping signals from the TICs. These pure compound peaks are then matched to the user-defined target library, using additional parameters of retention time and peak shape. There are options for parameter optimisation to select the optimal parameters. There is also the option to use internal standards for retention index, which applies penalties on the match factor for hits that have RIs very different to that of the reference compound.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Medium           | Package                               | Graphical User Interface                         | Windows                                               | -                                                 | 2.71                   | -                                                                                                                                                  | netCDF                                                                                           | Aglient, Bruker, Finnigan, INFICON, JEOL/Schrader, Kratos, Mach3, MassLynx, PerkinElmer, Schrader/GCMate, Varian, Xcalibur | 1996             | 2012               | http://chemdata.nist.gov/mass-spc/amdis/downloads/EULA-AMDIS.pdf                                                                                           | http://chemdata.nist.gov/dokuwiki/doku.php?id=chemdata:amdis                                                   | http://www.ncbi.nlm.nih.gov/pubmed/20185625                        | 21092268 |                                                                         |                                             |          |
| AMDORAP               | Accurate m/z detection method for LC-orbitrap (AMDORAP) provides preprocessing of MS data consisting of 3 stages: collect all data points with intensities above a threshold value, group them by m/z values and estimate representative m/z values for the clusters and extract EICs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-MS (Orbitrap)                  | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.0.6                  | xcms, mzR, Rcpp, BiocGenerics, Biobase                                                                                                             | mzXML, netCDF                                                                                    | -                                                                                                                          | 2011             | 2012               | GPL-2                                                                                                                                                      | http://amdorap.sourceforge.net/                                                                                | http://www.ncbi.nlm.nih.gov/pubmed/21702951                        | 19358599 |                                                                         |                                             |          |
| apLCMS                | This is an R package for the preprocessing of high resolution LC-MS data. It contains functions to reduce noise, identify features, correct for retention time deviation, align features across samples and recover weak-signals from LC-FTMS. First peaks are grouped based on their m/z, using a threshold value which is set using a mixture model. These groups are then split into smaller groups using kernel density functions by their rt and m/z. A run-filter is used to distinguish features from noise. The data are then grouped, with each group tentatively said to contain either one or a few features, with the aim of disentangling features with close m/z and rts. A kernel smoother is first fitted to estimate the number of features and their locations; this can return either a single peak, or if multiple peaks are found a maximisation-expectation (EM) algorithm with pseudo-likelihood is used for estimation. Next a feature table is generated for retention time alignment. Features are first aligned by m/z, using the same strategy as peak picking. A rt threshold value is then found using a model-based search. Pairwise rt differences between all features in a group are found and a triangular density function is to find the threshold. A kernel smoother is then used for rt alignment, using the profile with the most number of features as the template. Grouping is then reperformed. Weak signals from LC-FTMS data can then be recovered. Data is separated into XICs using adaptive binning. A large number of feature measurements are taken from every XIC. XICs are then matched to known metabolites; features mapped to known metabolites are more likely to be real signals. Machine learning techniques are used to find the optimum way of distinguishing between matched and unmatched XICs. Models are compared using cross validation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-FT-MS                          | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 6.2.0                  | MASS, rgl, mzR, splines, doSNOW, foreach, iterators, snow, ROCR, ROCS, e1071, randomForest, gbm                                                    | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2009             | 2016               | GPL (ł 2)                                                                                                                                                  | http://web1.sph.emory.edu/apLCMS/                                                                              | http://www.ncbi.nlm.nih.gov/pubmed/25005748                        | 25887958 |                                                                         |                                             |          |
| ChemoSpec             | This CRAN package provides functions for the chemometric analysis of NMR, IR or Raman spectroscopy data. It includes functions for spectral visualisation, peak alignment, HCA, PCA and model-based clustering. It is specifically designed for the analysis of metabolomics data, with the experimental design of treatment and control groups. Data should be in the .csv format with frequency values in the first column and intensity in the second. A standard pipeline would include: spectral inspection, baseline correction, alignment, removal of noise-only frequencies, binning and normalisation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | NMRIRRaman                     |                                   |                                         | Medium           | R Package                             | Command line interface                           | Linux, OS X, Windows                                  | R                                                 | 4.4.1                  | R (ł 3.1)                                                                                                                                          | .csv (first column intensity values, second column intensity values)                             | -                                                                                                                          | 2014             | 2016               | GPL-3                                                                                                                                                      | https://cran.r-project.org/web/packages/ChemoSpec/index.html                                                   | -                                                                  | 20626915 |                                                                         |                                             |          |
| cosmiq                | This bioconductor package provides preprocessing for LC-MS or GC-MS metabolomics or lipidomics data. It is designed to improve the detection of low abundance signals by generating a master map of all mz/RT space before the peak detection algorithm is applied. It is built using the xcmsSet object structure and so can be integrated with xcms (it is suggested to use retcor.orbiwarp for RT alignment). It uses a continuous wavelet transformation (CWT) peak detection algorithm adapted from Du et al 2006. It includes the "quality_combined"" function which is designed for use when there is no RT information available                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | so is suitable for peak picking of direct infusion experiments. "                                                                                                                                                                                                                                                                                   | Preprocessing                                                                                                                                                                                                                                                                                  |                                | GC-MSLC-MS                        |                                         | Untargeted       | Advanced                              | R Package                                        | Command line interface                                | Unix/Linux, Mac OS, Windows                       | R                      | 1.8.0                                                                                                                                              | R (ł 3.0.2), Rcpp                                                                                | mzData, mzML, mzXML, netCDF                                                                                                | -                | 2014               | 2016                                                                                                                                                       | GPL-3                                                                                                          | http://bioconductor.org/packages/release/bioc/html/cosmiq.html     | -        | -                                                                       |                                             |          |
| decoMS2               | This deconvolutes MS2 data using a novel two step process. Firstly MS2 peaks are collected with less stringent isolation settings. Spectra are then deconvoluted by evaluating MS2 fragment intensities as a function of retention time and precursor mass using the shifting window deconvolution approach.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.1                    | nnls, RColorBrewer, xcms, mzR, Rcpp, BioGenerics, Biobase, gplots                                                                                  | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2013             | 2013               | GPL-2                                                                                                                                                      | http://pattilab.wustl.edu/software/decoms2/decoms2.php                                                         | http://www.ncbi.nlm.nih.gov/pubmed/23829391                        | 24266674 |                                                                         |                                             |          |
| FeatureFinderMetabo   | This provides label-free quantification of metabolites of LC-MS data. It is available as part of OpenMS. There are two stages to the algorithm: mass trace detection and feature assembly. Each data set of peaks is sorted by decreasing intensity and peaks that are below a user-defined threshold are filtered. Each of the most intense peaks is then considered a potential seeding point for mass trace construction. Mass traces are features with a similar m/z, which occur in adjacent scans. The mass traces are extended along the retention time axis from the seeding point, in both directions. This results in more peaks with similar m/z being recruited into the mass trace, depending upon an intensity-weighted Gaussian model (that reflects that low-intensity peaks are less reliable than higher intensity peaks). The algorithm is aborted when a specified number of scans have been searched without finding an adequate peak. Once a peak has been added to a trace, it cannot be used as a seeding point or be added to another trace. At this stage the mass traces need to be separated, so that each trace only contains a single feature. LOWESS with a polynomial of degree 2 is used as a smoothing technique. It is then tested whether or not peaks are sufficiently separated in order to be considered separate peaks. If peaks need to be separated, the minima between the 2 chromatographic peaks is used as a splitting point. During feature assembly the aim is to find features originating from the same progenitor metabolite and to cluster these adducts. To identify isotopes a HiRes generated library of isotopic masses is used. A scoring function is used to assess how likely it is a set of mass traces are caused by the same metabolite based on precomputed mass difference distributions of potential metabolite compositions. A correlation similarity score is used to assess whether mass traces grouped by m/z are also compatible in their elution times (it must be at least 70% full width at half-maximum). For each hypothesis a combined score is calculated. To give preference to high intensity signals, the scores are weighted to peak area normalised by the total sum of mass traces. A list of features is then produced that should not contain multiple peaks for a single metabolite.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | Centroid LC-MS                    |                                         | Advanced/ Medium | Package                               | Command line interface/ Graphical User Interface | Unix/Linux                                            | C++                                               | 2.1.0                  | -                                                                                                                                                  | ConsensusXML, DTA, DTA2D, EDTA, featureXML, KROENIK, MGF, ms2, mzML, mzXML, mzData, PEPLIST, TSV | fid/XMASS                                                                                                                  | 2014             | 2016               | Three-clause BSD license                                                                                                                                   | http://ftp.mi.fu-berlin.de/pub/OpenMS/release-documentation/html/TOPP_FeatureFinderMetabo.html                 | http://www.ncbi.nlm.nih.gov/pubmed/24176773                        | 19040747 |                                                                         |                                             |          |
| flagme                | This bioconductor package processes and visualises GC-MS samples. It does not include support for peak detection (this can be done using AMDIS or ChromaTOF). It uses a dynamic programming-based alignment strategy - a similarity matrix is calculated based on a scoring function, taking into account similarity in RT and of the apex spectra, meaning that there are two components to the similarity: spectra and RT. A gap parameter must be set, with a high gap penalty discouraging gaps of peaks between spectra and a low gap penalty having very little cost for gaps (it is recommended to use 0.4-0.6). Similarly "D"" modulates the penalty for RT difference penalty.  Provides visualisation of total ion current (TIC) or extracted ion current (XIC) and 2D heat maps of intensity for individual samples. "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.30.0                 | gcspikelite, xcms, CAMERA                                                                                                                          | AMDIS.elu                                                                                        | ChromaTOF                                                                                                                  | 2014             | 2016               | LGPL (>=2)                                                                                                                                                 | https://bioconductor.org/packages/release/bioc/html/flagme.html                                                | -                                                                  | 25075114 |                                                                         |                                             |          |
| GSim                  | Designed for the visualisation and processing of NMR spectral data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | NMR                            | 1D NMR2D NMR                      |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | C++                                               | 0.20.1                 | -                                                                                                                                                  | ASCII                                                                                            | -                                                                                                                          | 2006             | 2013               | GPL- 2                                                                                                                                                     | https://sourceforge.net/projects/gsim/                                                                         | -                                                                  | 24711654 |                                                                         |                                             |          |
| HayStack              | A web-server for the visualisation and extraction of LC-MS features. Peaks can be extracted using XCMS. MetaboAnalyst can then be used for further preprocessing and statistical analysis: filtering, sum normalisation, log(2) transformation and autoscaling, followed by univariate (t-tests, volcano plots, ANOVA) and multivariate analysis (HCA, PCA, PLS-DA). TIC and BPC are automatically displayed and EIC can be generated. There is also a bin feature which splits full-scan MS data into uniform bins.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | MySQL, PHP, Perl, R                               | -                      | -                                                                                                                                                  | mzData, mzML, mzXML, netCDF                                                                      |                                                                                                                            | 2013             | 2014               | Free                                                                                                                                                       | http://binf-app.host.ualr.edu/haystack/                                                                        | http://www.ncbi.nlm.nih.gov/pubmed/25350247                        | 20977194 |                                                                         |                                             |          |
| HCOR                  | This provides retention time correction of LC-MS data, in particular reverse phase chromatography.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.01                   | xcms                                                                                                                                               | mzXML                                                                                            | -                                                                                                                          | 2014             | 2014               | GPL (ł 3)                                                                                                                                                  | http://b2slab.upc.edu/software-and-downloads/retention-time-drift-correction/                                  | -                                                                  | 22768229 |                                                                         |                                             |          |
| Ideom                 | IDEOM is an excel template which provides a GUI with implementations of mzMatch and xcms, along with macros for noise-filtering, metabolite identification and statistical analysis. msconvert can also be directly interfaced. Raw peaks can be extracted from within IDEOM or peak-picked data in the .txt or .csv format can be imported. Up to 10 internal/external standards can be entered and treatment/control groups can be entered for statistical analysis. Peaks are first picked with centWave and then mzMatch is used to convert them to .peakml. mzMatch can then be used to group peaks, filter peaks and annotate groups of related peaks. Normalisation can then be performed using TIC, median or used defined values (although this is not recommended for LC-MS). RT of authentic standards can be entered for metabolite identification. For metabolite identification, a metabolite library is also provided (consisting of KEGG, MetaCyc). To correct for mass calibration errors, a data-dependent polynomial mass recalibration step can be applied. There is then optionally tools for merging dual-polarity data, chemical formula determination for unidentified mass and stable isotope tracking. Specific metabolites can also be analysed in a targeted manner. Conditions can be statistically compared using univariate statistics: mean, relative intensity, SD, t-test and Fisher ratio.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Medium           | Excel Template                        | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | R                                                 | 19                     | Excel (ł 2007), R                                                                                                                                  | csv, mzXML, txt                                                                                  | Agilent .d, Thermo Fisher Scientific .raw, Bruker Daltonics .d, waters .raw                                                | 2012             | 2015               | LGPL-2                                                                                                                                                     | http://mzmatch.sourceforge.net/ideom.php                                                                       | http://www.ncbi.nlm.nih.gov/pubmed/22308147                        | 23111096 |                                                                         |                                             |          |
| iMet-Q                | Intelligent Metabolomic Quantitation (iMet-Q) is designed to provide accurate quantification of LC-MS data. Firstly peaks are detected. Optionally centroiding can be performed. Noise is removed by dividing scans into segments with 20% overlap (avoiding borderline issues), defining the noise level for each segment and removing signals below a user-specified S/N. The FWHM of each peak is calculated to automatically determine the chromatographic width. The charge state of a peak is determined by calculating the similarity between the monoisotopic peak and the second most abundant isotopic peak. Isotopic ratios are then determined for metabolite identification using the dot product ratio. Peaks are first aligned across technical replicates, then across different biological samples, using LOESS regression.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Easy             | Package                               | Graphical User Interface                         | Windows XP/7/8, Windows Server 2008/2012              | C#                                                | 1.001                  | ł .Net 4.0                                                                                                                                         | mzML, mzXML, netCDF                                                                              | -                                                                                                                          | 2015             | 2015               | LGPL-2                                                                                                                                                     | http://ms.iis.sinica.edu.tw/comics/Software_iMet-Q.html                                                        | http://www.ncbi.nlm.nih.gov/pubmed/26784691                        | 25566535 |                                                                         |                                             |          |
| intCor                | intCor provides functions for drift removal and data normalisation of LC-MS data. It accepts either raw data (netCDF or .mzXML), a peak matrix or an xcmsSet as input. It contains 5 methods for intensity drift correction: component correction, median fold change, ComBat and common PCA (CPCA). PCAs can also be performed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.03                   | R (ł 2.10), xcms, cpca, pcaPP, grid, gplots, sva, clValid, plyr, ggplot2, pcaMethods, methods, akima                                               | mzXML, netCDF, matrix, xcmsSet                                                                   | -                                                                                                                          | 2014             | 2014               | GPL-2                                                                                                                                                      | http://b2slab.upc.edu/software-and-downloads/intensity-drift-correction/                                       | http://www.ncbi.nlm.nih.gov/pubmed/24990606                        | 22833710 |                                                                         |                                             |          |
| LDA                   | Lipid Data Analyser (LDA) peak picks LC-MS lipidomics data by a novel algorithm designed for accurate quantification. Firstly a chromatograph is extracted and the time-point at peak summit is identified. Next a m/z range is extracted around this narrow time point. The peak summit is then redetermined and the border points of the profile peaks are detected and an ellipse fitted. It uses theoretical isotope distribution as an exclusion criteria. There is no deisotopic step. Three standardisation methods are available: based on a single standard, a median method and internally developed methodology (standardisation by most robust standards).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Lipidomics                              | Medium           | Package                               | Graphical user interface                         | Unix/Linux, Mac OS, Windows                           | Java                                              | 1.6.2                  | -                                                                                                                                                  | mzXML                                                                                            | Thermo Fisher Scientific .raw, waters .raw                                                                                 | 2010             | 2013               | Academic users only                                                                                                                                        | http://genome.tugraz.at/lda/lda_description.shtml                                                              | http://www.ncbi.nlm.nih.gov/pubmed/21169379                        | 24656939 |                                                                         |                                             |          |
| Maui-VIA              | Visualisation Identification and Alignment (VIA) provides simultaneous manual inspection and correction of identifications and alignments. It is based on Maui, an existing GC-MS processing software. It is designed to enable simultaneously process and correct assignments and alignments of GC-MS by providing a visualisation engine. Data should have been already peak picked and baseline corrected prior to import into Maui-VIA. Retention indices are calculated using linear interpolation. A user-imported metabolomics library in the .msp format can be supplied for feature annotation using a reverse search.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Medium           | Package                               | Gaphical user interface                          | Windows                                               | Java                                              | 1                      | -                                                                                                                                                  | mzML, netCDF                                                                                     | -                                                                                                                          | 2014             | 2014               |                                                                                                                                                            | http://bimsbstatic.mdc-berlin.de/kempa/software/kempaSoftware.html                                             | http://www.ncbi.nlm.nih.gov/pubmed/25654076                        | 17046977 |                                                                         |                                             |          |
| MeltDB                | This is a web application for metabolomics, providing functions for preprocessing, statistical analysis and deriving biological information from metabolomics data. It provides ontology based metabolomic experiment annotation. It provides analysis for both targeted and untargeted experiments. Peaks that have already been detected in other software can be imported, including those from vendor software. There is no enforced, set pipeline for analysis. Instead the user can choose the tools they wish to use. Peak picking can be performed using matchedFilter, centWave, MetaB or MassSpecWavelet. Retention indices can be calculated. Sum formula identification has also been implemented for annotation. The GMD or NIST are used for metabolite identification, using the dot product algorithm. For chromatographic alignment xcms, ChromA or a feature based alignment algorithm can be used. Statistical analysis available includes: Student's t-test, Kruskal-Wallis test, ANOVA, HCA, PCA, ICA, PLS-DA, Metabolite Correlation Analysis, Volcano plots, heatmaps, SVM and random forest. Bonferroni, Holm and Benjamini and Hochberg corrections can be calculated. Data can also be visualised in plots: TIC, EIC, MS/MS, alignment and spectral view.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     | GC-MS/MS                          |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Perl, Java, Javascript, MySQL, R                  | 2                      | -                                                                                                                                                  | AMDIS.elu, mzData, mzXML, netCDF                                                                 | Reports: Thermo Xcalibur (XLS), LECO ChromaTOF, MassHunter (Text), ChemStation peak list (Text)                            | 2008             | 2013               |                                                                                                                                                            | https://meltdb.cebitec.uni-bielefeld.de/cgi-bin/login.cgi                                                      | http://www.ncbi.nlm.nih.gov/pubmed/23918246                        | 22343432 |                                                                         |                                             |          |
| MET-COFEA             | MET-COFEA Metabolite compound feature extraction and annotation (MET-COFEA) provides a data analysis pipeline for LC-MS data, consisting of three modules: compound feature extraction, compound feature annotation and compound alignment. Mass traces are first extracted from each scan. The optimal mass trace candidate is then searched for. If two mass traces centroid masses are smaller than a user-defined threshold, they are merged. A continuous wavelet transformation (CWT) algorithm is then used for peak detection. To filter out bad peaks there are a number of options: peak intensity below a user-defined threshold, peak signal-to-noise ratio, peak significance level, zig-zag index and triangle peak area similarity ratio (TPASR). Next features are annotated with adducts and isotopes. After annotations, peaks are then grouped by dot-product to the highest intensity peaks by retention times and peak similarity. Compounds that fall into the same group, due to coelution and similar peak shapes are then separated using accurate mass information. There is also a peak refinement algorithm to separate peaks that cannot be separated by accurate mass (e.g. adduct peak intensities were too weak to be detected). HMDB is used for reference compound libraries to identify these peaks. Compounds are then aligned using a novel compound alignment strategy based on intensity ranks rather than elution time.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Medium           | Package                               | Gaphical user interface                          | Windows                                               | C++, .NET                                         | Gamma                  | -                                                                                                                                                  | netCDF                                                                                           | -                                                                                                                          | 2013             | 2016               |                                                                                                                                                            | http://bioinfo.noble.org/manuscript-support/met-cofea/                                                         | http://www.ncbi.nlm.nih.gov/pubmed/24856452                        | 22238263 |                                                                         |                                             |          |
| MET-COFEI             | Metabolite compound feature extraction and identification (MET-COFEI) is a data processing platform for GC-MS data. It consists of three modules: compound feature extraction, compound identification and compound alignment. The compound feature extraction module itself consists of three submodules: EIC extraction, peak detection and peak filtering. Compound identification also consists of three submodules: peak grouping, pure spectrum reconstructing and library searching. The meaningful mass traces are first extracted with EIC extraction. Peak detection is then performed for each of these mass traces. Poor quality peaks are then filtered out with peak filtering. Peaks with similar retention time and peak shape are then clustered with peak grouping. All the m/z intensity pairs at the apex position in the same group are then combined to reconstruct a pure spectrum. A user specified library (.msl) is then searched for metabolite identification. Compounds are then aligned across samples.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Medium           | Package                               | Gaphical user interface                          | Windows                                               | C++                                               | Beta                   | -                                                                                                                                                  | netCDF                                                                                           | -                                                                                                                          | 2014             | 2014               |                                                                                                                                                            | http://bioinfo.noble.org/manuscript-support/met-cofei/index.php?page=Introduction                              | -                                                                  | 27452369 |                                                                         |                                             |          |
| MET-IDEA              | Metabolomics Ion-based Data Extraction Algorithm (MET-IDEA) performs preprocessing and metabolite identification of GC-MS data. It can also be used for LC-MS, but it is not optimised for these. It is designed to take AMDIS output as input and can quantify the results. As input it takes a list of ions and their retention times. From these it generates mass spectral tags. A user inputed list containing a mass spectral library can be used for metabolite identification. Optionally mass and/or retention time values can be calibrated by user selected markers.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS LC-MS                    |                                   | Targeted                                | Advanced         | Package                               | Command line interface                           | Windows                                               | .NET                                              | 2.08                   | -                                                                                                                                                  | AMDIS.elu, netCDF                                                                                | -                                                                                                                          | 2006             | 2012               |                                                                                                                                                            | http://bioinfo.noble.org/download/                                                                             | http://www.ncbi.nlm.nih.gov/pubmed/16808440                        | 26834843 |                                                                         |                                             |          |
| MET-XAlign            | MET-XAlign works alongside MET-COFEA. It is aimed that MET-COFEA will produce an annotated list of metabolites (characterised by neutral molecular mass and estimated retention time). MET-XAlign will then align compounds across different experimental conditions, if the neutral molecular mass and estimated retention time fall within an acceptable tolerance window. There are two alignment strategies: fast and deep. MET-COFEA parameters are configured differently for different experimental conditions to optimise metabolite alignment across samples from different experiments using MET-XAlign.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Medium           | Package                               | Graphical user interface                         | Windows                                               | C++, .NET                                         | Beta                   | MET-COFEA                                                                                                                                          | MET-COFEA output                                                                                 |                                                                                                                            | 2015             | 2015               |                                                                                                                                                            | http://bioinfo.noble.org/manuscript-support/met-xalign/                                                        | http://www.ncbi.nlm.nih.gov/pubmed/26247233                        | 23494783 |                                                                         |                                             |          |
| Metab                 | This bioconductor package is designed for the analysis of GC-MS data preprocessed with AMDIS (Automated Spectral Deconvolution and Identification System). AMDIS deconvolutes spectra and provides metabolite identification via spectral libraries, however it has a number of limitations for use in metabolomics, as it outputs results as a spreadsheet per sample (or a merged sheet which requires a lot of manual processing before statistical analysis). It also has limited use in metabolite quantification (as it uses the area under the peak to do this, which may be influenced by the coelution of different metabolites) and may report more than one metabolite identified for the same mz/RT. Metab works to solve these limitations by selecting the most probable metabolite, correcting abundance calculation and presenting results in a single spreadsheet. It takes the AMDIS report and raw data in the netCDF format as input, requiring both for the function MetReport. It also has functions to remove false positives, normalise data by an internal standard, normalise by biomass and perform ANOVA or t-Test.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.8.0                  | R (ł 3.0.1) xcms, svDialogs                                                                                                                        | AMDIS.elu, netCDF                                                                                | -                                                                                                                          | 2011             | 2016               | GPL (ł 2)                                                                                                                                                  | https://www.bioconductor.org/packages/release/bioc/html/Metab.html                                             | http://www.ncbi.nlm.nih.gov/pubmed/21697128                        | 22851531 |                                                                         |                                             |          |
| MetaboliteDetector    | This provides preprocessing and metabolite identification of both targeted and non-targeted GC-MS data. For TOF data, a calibration function is included, to calibrate from TOF to m/z. The baseline of each spectra is first corrected, followed by applying a five point Savitzky-Golay filter to smooth the data if noise is present. Single ion peaks in the chromatogram are then detected using the first derivative of the smoothed intensity values and deconvolution is performed using an improved version of the algorithms applied by Colby et al. and Stein. After this, the m/z of potential compounds are extracted. It is recommended that the RI for each compound is calculated for metabolite identification. The user can provide a library of RIs for metabolite identification. Quantification is then performed and chromatographs are aligned by RT. There is also the option to perform PCA.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   | Targeted,Untargeted                     | Medium           | Package                               | Command Line Interface/ Graphical User Interface | Unix/Linux, Mac OS, Windows                           | C++                                               | 2.5                    | -                                                                                                                                                  | netCDF                                                                                           | JEOL FastFlight2                                                                                                           | 2009             | 2014               | GPL                                                                                                                                                        | http://md.tu-bs.de/                                                                                            | http://www.ncbi.nlm.nih.gov/pubmed/19358599                        | 25348215 |                                                                         |                                             |          |
| MetaDB                | MetaDB is an open-source web application for Metabolomics metadata management and data processing. It is based on ISA tab as metadata input format. The analysis of untargeted data is done using the R package MetaMS. This software is a project of Fondazione Edmund Mach.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   |                                         | Advanced         | R Package                             | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | R                                                 | -                      | Java 1.7, Tomcat 6, MySQL 5.5, Grails 2.2.3, R 3.0.1                                                                                               |                                                                                                  | -                                                                                                                          | 2013             | 2014               | CPAL License, MIT License                                                                                                                                  | https://github.com/rmylonas/MetaDB                                                                             | http://www.ncbi.nlm.nih.gov/pubmed/25566535                        | 25692814 |                                                                         |                                             |          |
| MetAlign              | MetAlign provides preprocessing functions including noise reduction, missing value imputation, baseline correction, peak-picking and retention time alignment. Initially all data is converted to the netCDF format; during this conversion binning is performed. There is the option to process either nominal or accurate mass spectral data (but accurate requires a second baseline correction). There are two modes of alignment: respectively rough and iterative. A user defined windows runs through the time dimension in rough mode, with peaks within a mass trace being grouped by amplitude. Iterative mode uses the same algorithm as rough, but after, peaks that are present in all data sets are selected as landmarks. The number of landmark peaks present above a certain amplitude in a mass trace is then calculated. The average time dimension difference for each dataset is then calculated and this is used as input for the next iteration. The iterative cycle ends when the moving time window is in the order of a mass peak width.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   | Untargeted                              | Advanced         | Package                               | Command line interface                           | Windows 7 (32 and 64 bit)/XP/NT/2000                  | Visual C++                                        | 3                      | -                                                                                                                                                  | mzData, mzXML, netCDF                                                                            | Agilent .d, Masslynx .raw, Xcalibur .raw                                                                                   | 2003             | 2012               |                                                                                                                                                            | http://www.wageningenur.nl/en/show/MetAlign-1.htm                                                              | http://www.ncbi.nlm.nih.gov/pubmed/22833710                        | -        |                                                                         |                                             |          |
| metaMS                | This bioconductor package is based on xcms and CAMERA but with a number of differences as it is adapted for GC-MS analysis. Unlike xcms, it performs a pseudospectra based analysis, rather than a feature based analysis, which avoids the alignment stage (which can be difficult in GC-MS data). There is also the ability to create a database of in-house standards.  It also contains a wrapper function of xcms and CAMERA for LC-MS analysis and the ability to create a database of standards for automatic annotation of these metabolites.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.6                    | R (ł 2.10.0) methods, xcms (ł 1.35.0), CAMER0                                                                                                      | mzXML, netCDF                                                                                    | -                                                                                                                          | 2014             | 2015               | GPL (ł 2)                                                                                                                                                  | https://www.bioconductor.org/packages/release/bioc/html/metaMS.html                                            | http://www.ncbi.nlm.nih.gov/pubmed/24656939                        | 24261687 |                                                                         |                                             |          |
| MetExtract            | This is designed for the analysis of isotopically labelled mass spectrometry data. It aims to elucidate signals of natural and stable isotopically labelled metabolites and combine these, to allow assignment of the biological metabolites. For this the experimental design must include two samples, that only differ in isotopic labelling of one specific element, with one being labelled and the other not. Initially all peaks above a certain threshold are considered to be the monoisotopic peak of a non-labelled metabolite. A defined range of atom numbers is then used to find the corresponding labelled isotopologue. Pearson's Correlation Coefficient is then calculated for the EIC of both labelled and unlabelled molecules. Only if the isotopic peaks for both are found is the peak added to the m/z list. The picked m/z values are then binned according to m/z and rt similarity using hierarchical clustering. This does not separate structural isomers. The binned data are then searched for adducts and fragments using an m/z value offset list.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | Centroid LC-MS                    | Untargeted, Isotopic Labelling Analysis | Advanced         | Package                               | Command line interface                           | Windows (Full binaries only available for Windows XP) | C++                                               | 1                      | -                                                                                                                                                  | mzXML                                                                                            | -                                                                                                                          | 2011             | 2012               | MIT License                                                                                                                                                | https://code.google.com/archive/p/metextract/                                                                  | http://www.ncbi.nlm.nih.gov/pubmed/22238263                        | -        |                                                                         |                                             |          |
| MetTailor             | This is designed for use after initial peak picking and alignment but before statistical analysis. It provides dynamic block summarisation (DBS) to correct for peak misalignments and isotopic pattern validation to verify correct alignments. DBS is used to create blocks which capture a single compound, which is present in the majority or all samples. Each block is uniformly sized. A peak is considered well aligned if it is present above a given intensity, in above a threshold number of samples. A peak is missing if it not present/ below an intensity threshold. If a given block contains no well aligned peaks, DBS identifies a lead peak (sample with smallest number of missing data), if current block centre coincides with this peak, the peak is then marked as well aligned. If not the centre is then placed to the lead peak and repeated. A quantitative summary is then captured for each block from the peak apex or largest integrated peak area. This summary can then be used to correct misalignments, resulting in fewer missing data. There is then the option to extract isotopic patterns to ensure that the alignments are correct, by using natural isotope distributions. This involves determining the charge of the candidate compound and extracting the ratio between the monoisotopic and isotopic peaks. RT(delta)-based normalisation can be used to correct the effect of temporal factors. It uses the weighted sum of peak area values in the RT neighbourhood for normalisation. Alternatively total intensity sum (TIS) normalisation can be used (TIS is calculated and the data is rescaled to the TIS across all compounds).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 2                      | -                                                                                                                                                  | .txt/ .csv                                                                                       | -                                                                                                                          | 2015             | 2015               | GPL (ł 2)                                                                                                                                                  | https://sourceforge.net/projects/mettailor/                                                                    | http://www.ncbi.nlm.nih.gov/pubmed/26220962                        | 26220962 |                                                                         |                                             |          |
| MRM-DIFF              | Multiple reaction monitoring-based differential (MRM-DIFF) allows for the analysis of lipidomics LC-MS MRM assays. It relies on pooled QC samples, for the LOES/cubic spline method. From all the pooled QC samples, one is chosen as a reference (this is the chromatograph whose gravity is closest to the midpoint). Using correlation optimized warping (COW), all other chromatograms are aligned to this reference. A user defined library of retention time, MRM transitions and molecular formulas is used for metabolite identification and to estimate isotopes. A peak detection algorithm that uses the abundance of detected peaks for multivariate analysis is implemented. Isotopic peak abundance is resolved and signal intensity drifts are corrected using LOESS and cubic spline normalisation. Alternatively metabolite concentrations can be normalised to an internal standard. Peaks are detected by first finding the median amplitude (AF), first-order derivative (FF) and second-order derivative (SF) (the derivatives are calculated using Savitzky-Golay filter). The maximum amplitude difference between two adjacent peaks is detected, as well as the maxima between the first and second order derivatives. The edges of the peaks are recognised when the amplitude and first order derivative are both greater than AF and FF in two adjacent points and the tip is recognised when the sign of the first derivative changes and the second order derivative is less than SF. Metabolite identification is based on retention time accuracy. To calculate isotopic abundances and annotate peaks, the theoretical isotopic ratio of the molecular formula is calculated.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-QqQ-MSLC-QqQ-MS/MS             | Lipidomics                              | Basic            | Package                               | Graphical User Interface                         | Windows                                               | C#                                                | -                      | ł .NET Famework 4.0, ł4GB RAM                                                                                                                      | Reifycs .ABF, .mzML                                                                              |                                                                                                                            | -                | 2014               |                                                                                                                                                            | http://prime.psc.riken.jp/Metabolomics_Software/MRM-DIFF/index.html                                            | http://www.ncbi.nlm.nih.gov/pubmed/25688256                        |          |                                                                         |                                             |          |
| MRMPROBES             | Multiple reaction monitoring based probabilistic system for widely targeted metabolomics (MRMPROBES) is a RIKEN software that provides automatic detection and annotation of fragments from MRM experiments, as well as statistical analysis. Firstly peaks are detected and smoothed and peak groups are found. Peaks are detected by first finding the median amplitude (AF), first-order derivative (FF) and second-order derivative (SF) (the derivatives are calculated using Savitzky-Golay filter). The maximum amplitude difference between two adjacent peaks is detected, as well as the maxima between the first and second order derivatives. The edges of the peaks are recognised when the amplitude and first order derivative are both greater than AF and FF in two adjacent points and the tip is recognised when the sign of the first derivative changes and the second order derivative is less than SF. Metabolite peaks are then evaluated by posterior probability (the odds ratio in multivariate logistic regression) using: peak intensity, retention time, qualifier/target ratio, shape and coelution similarity, with each peak being assigned a score. The Gaussian function is used to assess retention time and QT ratio, peak intensity is assessed by comparing the height of the peak to the highest peak of all transition records and the rank of the peak intensity in a focused transition record and shape and coelution were assessed with the mProphet algorithm. The qualifier/target ratio is used for the assessment of peak groups, where the target is the transition which is used to quantify the metabolite and the qualifier transitions are the remaining transitions used for qualification (discrimination from background noise and isomeric metabolites). Peak groups with the highest probability are then selected as the quantification value. During this process, metabolites are identified by comparison to a reference library. The metabolite intensities can then be normalised to an internal standard using the LOESS of the second-degree least-squares with a robust tri-cubic weight function. Statistical analysis can then be performed with PCA and multi t-tests being provided.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     | GC-QqQ-MSLC-QqQ-MS                |                                         | Basic            | Package                               | Graphical user interface                         | Windows                                               | C#                                                | 2.3                    | ł .NET Famework 4.0                                                                                                                                | Reifycs .ABF, .mzML                                                                              | Agilent .d, AB Sciex .wiff, Thermo Fisher Scientific .raw, Shimadzu .LCD                                                   | 2013             | 2016               |                                                                                                                                                            | http://prime.psc.riken.jp/Metabolomics_Software/MRMPROBS/index.html                                            | http://www.ncbi.nlm.nih.gov/pubmed/23581547                        | 22111688 |                                                                         |                                             |          |
| MS-DIAL               | Provides deconvolution for untargeted data-independent acquisition (DIA) tandem MS data. Initially data are converted into the Analysis Base File (ABF) format, to allow for rapid data extraction. For peak detection, the peak detection algorithm first starts with a smoothing method (linearly weighted smoothing average, moving average, Savitzky-Golay or binomial filter) for retention time and accurate mass. Peaks are then detected by first finding the median amplitude (AF), first-order derivative (FF) and second-order derivative (SF) (the derivatives are calculated using Savitzky-Golay filter). The maximum amplitude difference between two adjacent peaks is detected, as well as the maxima between the first and second order derivatives. The edges of the peaks are recognised when the amplitude and first order derivative are both greater than AF and FF in two adjacent points and the tip is recognised when the sign of the first derivative changes and the second order derivative is less than SF. Two dimensional peak detection or peak spotting occurs by using retention time and accurate mass (MS1). Detected peak tops are displayed as spots, with spots of the same retention time and similar m/z being compared by peak height, to evaluate whether they should be merged or not. Each peak then undergoes deconvolution using the MS^2dec algorithm, which first extracts the product spectra for all precursor peaks on all MS/MS spectra (based on GC-MS deconvolution, but using accurate mass instead of nominal mass). Least squares optimisation is used to extract model peaks from chromatograms. Background noise and coeluted metabolites are removed by determining peak heights of reconstructed chromatograms. After smoothing, the chromatograph undergoes baseline correction by finding the local minimum of a user defined segment, calculating the median of all local minimums and discarding points above this value. Ideal slope and sharpness values are calculated, with an ideal slope of >0.95 required for a peak to be considered a model peak. Metabolites are identified using MassBank, LipidBlast, NIST or a custom list. Metabolite identifications are assigned a score based on similarity of MS/MS, MS1, RT and isotope ratio. Peak alignment is performed using an algorithm based on Joint Aligner in MZmine. Missing value imputation is also provided. Sequential windowed acquisition of all theoretical (SWATH) acquisition is a DIA approach, which was particularly used to test this software.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-MS/MS                          | Untargeted                              | Advanced         | Package                               | Command line interface, Graphical user interface | Windows XP/Vista/7/8                                  | C#                                                | 2.5                    | ł .NET Famework 4.0, ł4GB RAM                                                                                                                      | .mzML                                                                                            | Agilent .d, AB Sciex .wiff, Thermo Fisher Scientific .raw, Bruker Daltonics .d, waters .raw                                | 2015             | 2016               |                                                                                                                                                            | http://prime.psc.riken.jp/Metabolomics_Software/MS-DIAL/index.html                                             | http://www.ncbi.nlm.nih.gov/pubmed/25938372                        | 23039714 |                                                                         |                                             |          |
| MZmine                | MZmine provides a toolkit of modules for MS data analysis. It now includes modules for: raw data import, raw data processing, peak detection, peak list alignment, peak identification, visualisation, normalisation and statistical analysis. It is able to process profile, centroid and MSn data. There are a number of different algorithms for peak detection: Local maxima - which detects the local maxima of the spectrum, Recursive threshold - which reduces false positive by not detecting noise peaks, Wavelet transform - which processing each spectra by matching them to the Mexican hat wavelet model using continuous wavelet transformation, Exact Mass - for high quality spectra, which uses FWHM to determine the centre of each peak, Centroid - for centroided data, which detects all peaks above a specified noise level and GridMass. There is an optional filter for FT-ICR-MS data that builds a theoretical model (Gaussian or Lorentzian) with a given mass resolution around each peak and removes noise below this model. After this m/z features are connected into a chromatograph. Individual peaks are then deconvoluted with a number of algorithms provided for this purpose: Baseline cut-off - peaks must have intensity and time spans over given ranges, Noise amplitude - the same as Baseline cut-off but the baseline is initially set to the level where the noise is most concentrated and Local minimum search identifies the local minima as the borders between peaks. There is an extra module that allows the noisy deconvoluted data to be fit to an ideal peak model. For alignment the RANSAC (random sample consensus) algorithm is used. It uses the RANSAC and alignment windows, consisting of the same m/z threshold but different rt thresholds. A scatterplot of all candidate alignments is made and the RANSAC algorithm is used to build a candidate model of alignment. LOESS is then applied and the correct retention time shift is predicted and then the peaks are then joined. The old Join aligner algorithm is also available. A number of options are then available: missing data filling, isotope detection, filtering and normalisation. For peak annotation, CAMERA can be used. Peaks can either be identified via a custom database or by searching online databases (PubChem, KEGG, METLIN, HMDB, ChemSpider, PlantCyc, LipidMaps and MassBank all directly interface with MZmine). NIST can also be directly interfaced. For statistical analysis a number of options are available: PCA, clustering, heatmaps, curvilinear distance analysis (CDA), Sammon's Projection and Log Ratio Plots. GridMass is a new feature detection algorithm for HRLC-MS data. LC-MS data can be represented on a 2D plot of m/z against retention time, with colour matching intensity. A grid of equally spaced probes is generated to cover the entire area. Each probe then finds the local maximum and it is then moved to this location to search for a higher intensity value. This process is repeated until no higher value can be detected, and the current probe location is then marked as a peak. | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS CE-MSGC-MS               | LC-MS/MS                          | Untargeted                              | Medium           | Toolkit                               | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | Java, R                                           | 2.23                   | JRE ł 1.7, R, Rserve, ptw, gplots, baseline, hyperSpec, ggplot2, xcms, CAMERA, PROcess                                                             | mzData, mzML, mzXML, netCDF                                                                      | Thermo Fisher Scientific .raw, waters .raw, Aglient .csv                                                                   | 2005             | 2016               | GPL-2                                                                                                                                                      | http://mzmine.github.io/                                                                                       | http://www.ncbi.nlm.nih.gov/pubmed/20650010                        | 25803548 |                                                                         |                                             |          |
| Newton                | This program can either be used locally or as a web-based application. It is specifically designed for the analysis of 2D 1H-13C HSQC spectra, but can also be applied to 1D 1H and 2D 1H-TOCSY spectra. The fast maximum likelihood reconstruction (FMLR) is used to perform spectral deconvolution iteratively. A data ensemble of three distinct spectra is constructed: a data spectrum acquired from conventional Fourier transformation  of the acquired FID, a model spectrum obtained from the sum of the cropped signals of a synthetic time domain along each dimension and a residual spectrum which is the difference between the data and model spectra. It allows input of data directly from NMRPipe.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | NMR                            | 2D 1H, 13C-HSQC NMR               |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | Java                                              | 1.4.10                 | Java ł 1.6                                                                                                                                         | UCSF (sparky)                                                                                    | -                                                                                                                          | 2013             | 2013               |                                                                                                                                                            | http://newton.nmrfam.wisc.edu/                                                                                 | -                                                                  | 25688256 |                                                                         |                                             |          |
| Nmrglue               | This provides a number of functions for NMR data preprossing including: apodization, spectral shifting, Fourier transformation, baseline smoothing and flattening, and linear prediction modeling and extrapolation. It is not specificially desiigned for metabolomics.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | NMR                            |                                   |                                         | Advanced         | Package                               | Command line Interface                           | Unix/Linux, Mac OS, Windows                           | Python                                            | 0.6-dev                | NumPy, SciPy                                                                                                                                       | NMRPipeUCSF (sparky)                                                                             | BrukerVarian/Agilent                                                                                                       | 2013             | 2016               | BSD                                                                                                                                                        | https://www.nmrglue.com/                                                                                       | https://www.ncbi.nlm.nih.gov/pubmed/23456039                       | 23581547 |                                                                         |                                             |          |
| RAMClustR             | This is designed for the analysis of indiscriminate (data-independent-acquisition) MS/MS data that provides a feature grouping method. It takes a dataset preprocessed with XCMS as input (it can take data preprocessed by other software as input, but the user then needs to manually input parameters that are calculated automatically from the xcmsSet). Firstly quantile normalisation is available. If MS/MS data is present, to separate the MS and MS/MS the user can either tag the filename or file path. A custom similarity matrix using two Gaussian terms (one of the different between rt of two features and the other the correlation between two features across all samples) is used to group features. Pearson's correlation is used to calculate similarity. Features that are similar in both Gaussian terms are clustered using a similarity matrix.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-MS/MS                          |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.2                    | devtools                                                                                                                                           | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2013             | 2016               | GPL-2                                                                                                                                                      | https://github.com/cbroeckl/RAMClustR                                                                          | http://www.ncbi.nlm.nih.gov/pubmed/24927477                        | 25938372 |                                                                         |                                             |          |
| RANSY/RAMSY           | Ratio analysis of mass spectrometry (RAMSY) aims to remove the effect of interfering peaks and simplify the spectra. It is based on RANSY (Ratio analysis of nuclear magnetic resonance spectroscopy). It is calculated by calculating the ratios between a driving peak and all the other features in a spectra. It is available as a web app, matlab script and excel macro.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS NMR                 |                                   |                                         | Advanced         | Package/ Web App                      | Command line interface, Web user interface       | Unix/Linux, Mac OS, Windows                           | MATLAB/ Excel                                     | 2                      | -                                                                                                                                                  | Peaklist                                                                                         | -                                                                                                                          | 2013             | 2013               |                                                                                                                                                            | http://depts.washington.edu/nwmrc/RAMSY                                                                        | http://www.ncbi.nlm.nih.gov/pubmed/24168717                        | 23584940 |                                                                         |                                             |          |
| rNMR                  | Designed for analysis 1D and 2D NMR spectra using a ROIs based approach (which contain all of the underlying NMR data), rather than the traditional peak lists. ROI can be visually inspected to help aid accurate quantification. It contains a convert function, which will convert raw NMR files into a format that rNMR can analyse. It contains 3 automatic peak picking buttons: Full, Region and ROI. The produced peak lists can then be directly exported to MMCD or uploaded to BMRB for identification.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | NMR                            | 1D NMR 2D NMR                     |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X, Windows                                  | R                                                 | 1.1.9                  | R (ł 2.10.0)                                                                                                                                       | UCSF (sparky)                                                                                    | -                                                                                                                          | 2011             | 2015               | GPL-3                                                                                                                                                      | http://rnmr.nmrfam.wisc.edu/                                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/19821464                        | 25263576 |                                                                         |                                             |          |
| SIMAT                 | This bioconductor package performs preprocessing and metabolite identification of selective ion monitoring (SIM) GC-MS data. Firstly, then most unique fragment (optimal fragment) must be manually selected. Next, baseline correction is performed on the individual fragments. Next a smoothing filter is used. Peaks are then detected by finding all candidate peaks in the vicinity of the expected RT and evaluating them. Each EIC is then evaluated for its quality, to decide whether or not it is included when estimating the similarity score or the peak intensity. RI calibration is then performed, using a piecewise linear regression model of RT - it is recommended to use RI standards for this. For feature identification a mixed score of the weighted dot-product of the spectra and average pairwise insensity comparisons between 2 spectra using the ratios of all fragments. The user can choose AUC-based or apex-based scores. The intensity of each fragment is then calculated using both the AUC and the apex. Both the TIC and EICs can be visualised using SIMAT. This is also available as a MATLAB script.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          | GC-MS - SIM                       | Targeted                                | Advanced         | R Package/ MATLAB Script              | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R/ MATLAB                                         | 1                      | R (ł 3.0.0), Rcpp (ł 0.11.3)                                                                                                                       | netCDF, target peak list                                                                         | -                                                                                                                          | 2015             | 2016               | GPL-2                                                                                                                                                      | http://bioconductor.org/packages/3.1/bioc/html/SIMAT.html                                                      | http://www.ncbi.nlm.nih.gov/pubmed/26283310                        | 22833709 |                                                                         |                                             |          |
| SOAP-NMR              | SOAP-NMR provides many functions for the processing of 1H NMR metabolomics data. Functions provided include apodization, baseline correction, bucketing, Fourier transformation, warping and phase correction. Bruker FID can be directly imported.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1                      | R (ł 2.10)                                                                                                                                         | -                                                                                                | Bruker FID                                                                                                                 | 2015             | 2016               | GPL-2                                                                                                                                                      | https://github.com/ManonMartin/SOAP-NMR                                                                        | -                                                                  | 22782550 |                                                                         |                                             |          |
| SpectConnect          | This is designed for the detection of features in GC-MS data without the need to use a reference compound library. Every spectrum is compared to every other spectrum using the Gemoda algorithm, which finds pairwise similarity of clusters (cliques) using weighted dot product. The most representative spectra for each clique is chosen. This allows it to identify which features are preserved across samples. It can also find peaks which differentiate between conditions. It requires that the user provides sample replicates (technical, but it is also desirable to include biological).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Basic            | Web App                               | Web User Interface                               | Unix/Linux, Mac OS, Windows                           | Python, C                                         | -                      | -                                                                                                                                                  | AMDIS.elu                                                                                        | -                                                                                                                          | 2007             | 2012               |                                                                                                                                                            | http://spectconnect.mit.edu/                                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/17263323                        | 27419259 |                                                                         |                                             |          |
| TargetSearch          | This Bioconductor packages provides both preprocessing and metabolite identification of GC-MS data: providing peak picking, RT alignment and metabolite identification via libraries. Features are picked using peak apex intensities rather than deconvolution as deconvolution can induce errors resulting in mixed mass spectral tags (MSTs). As input it requires raw data in the netCDF format (ideally pre-baseline corrected), a text file of metadata and a reference library of MSTs. For RT correction to be performed a tab separated file of retention time index (RI) markers must be provided. If baseline correction was not performed using the vendor software, this functionality is provided in TargetSearch. After this, peak apex intensities are used for peak picking. Retention time alignment is then performed using the RI. Metabolites can then be identified by user inputed libraries.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   | Targeted                                | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R, C                                              | 1.2.0                  | R (ł 2.7.0), mzR                                                                                                                                   | netCDF                                                                                           | -                                                                                                                          | 2009             | 2016               | GPL (ł 2)                                                                                                                                                  | http://bioconductor.org/packages/2.5/bioc/html/TargetSearch.html                                               | http://www.ncbi.nlm.nih.gov/pubmed/20015393                        | 24174567 |                                                                         |                                             |          |
| X13CMS                | This is an extension of XCMS that provides support for isotopic labelling. It requires two biologically identical samples to be prepared, with isotopic labelling being applied to one of them. XCMS is used for peak detection and retention time alignment, with the table of putative detected peaks being forwarded to X13CMS. X13CMS function getIsoLabelReport() can then be used to detect which metabolites have been enriched with isotopic labelling. This function detects the difference in mass between labelled and unlabelled samples and outputs a table of isotopologues that have statistically significant isotopic labelling. It requires that the mean intensity of the labelled peak be less than the unlabelled peak. getIsoDiffReport() then compares labelling between different biological conditions, to find significant biological differences between them (using a Welch's t-test).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.4                    | xcms, mzR, Rcpp, BioGenerics, Biobase                                                                                                              | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2014             | 2014               |                                                                                                                                                            | http://pattilab.wustl.edu/software/x13cms/x13cms.php                                                           | http://www.ncbi.nlm.nih.gov/pubmed/24397582                        | -        |                                                                         |                                             |          |
| XCMS                  | XCMS is a bioconductor package that provides preprocessing for LC-MS (and GC-MS and DI-MS) data. The following pipeline is used for preprocessing: peak picking, peak grouping, retention time alignment, peak regrouping and filling in missing values. XCMS currently provides five peak detection algorithms: matchedFilter and centWave are established methods; Massifquant, MS1-methods and MSW-methods have been more recently released. These different methods are optimised for different experimental setups. CentWave uses continuous wavelet transformation to pick peaks in centroid format high resolution LC-MS data, matchFilter is peck detection of low resolution data, Massifquant uses a 2D kalman filter for high resolution LC-MS peak picking (isotope trace detection, using the TracMass algorithm), MS1-methods are used to ensure the detection of MS1 precursor peaks, when MS1 and MS2 data are gathered quasi simultaneously, MSW-methods are for peak detection of DI-MS data (it is a wrapper of peak picking algorithm from the ŇMassSpecWaveletÓ package). Three methods for peak grouping are provided: group.density (peaks are grouped across samples into overlapping m/z bins), group.mzClust (high resolution alignment) and group.nearest (a master peak list is created and corresponding peaks from all samples are assigned - inspired the mzMine grouping algorithm). For retention time correction the methods available are: loess and obiwarp (Ordered Bijective Interpolated Warping). There are two methods for filling in missing data: one for chromatographic spectra, one for DI-MS. This is done by integrating the values of a peak group. It is also possible to analysis LC-MS/MS data with XCMS.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     | LC-MS/MS                          | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R, C++                                            | 1.50.1                 | mzR, BiocGenerics, ProtGenerics, Biobase                                                                                                           | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2005             | 2016               | GPL (ł 2) and file license                                                                                                                                 | http://bioconductor.org/packages/release/bioc/html/xcms.html                                                   | http://www.ncbi.nlm.nih.gov/pubmed/16448051                        | -        |                                                                         |                                             |          |
| xMSanalyzer           | Designed to improve feature detection of LC-MS data, running in conjunction with XCMS or apLCMS. As the scanning intervals with Fourier transform mass spectrometers is relatively constant, the optimum number of points to define a peak can vary over the course of a run. To deal with this, xMSanalyzer selects the number of points that is most reproducible for each peak, allowing for the extraction of low intensity peaks and those found in only a few samples. Sample quality is then evaluated using mean pairwise Pearson correlation coefficient between sample replicates. Quantitative feature reproducibility across sample replicates can then be evaluated using percent intensity difference (PID) or coefficient of variation (CoV). There is also the option of merging features detected using different parameter settings. Features can be matched across samples based on their m/z and their retention time. Features that are present in only single/few biological replicates but across multiple technical replicates can also be detected. It annotate metabolites using the METLIN and KEGG databases.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Preprocessing                                                                                                                                                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 2.0.6.1                | Rcpp, mzR, R2HTML, XML, limma, xcms, snow, rgl, gplots, foreach, RCurl, doSNOW, apLCMS                                                             | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2013             | 2015               | GPL-2                                                                                                                                                      | https://sourceforge.net/projects/xmsanalyzer/                                                                  | http://www.ncbi.nlm.nih.gov/pubmed/23323971                        | 24576144 |                                                                         |                                             |          |
| Bayesil               | The web system, Bayesil is designed to supply automatic spectral processing and identification of serum, plasma and cerebrospinal fluid (CSF) 1D 1H NMR spectra. Automated zero-filling, Fourier and Hilbert transformation, phasing, baseline correction and solvent filtering, smoothing, chemical shift referencing and reference deconvolution. A reference compound with known concentration, is then used for quantification. The spectral library provided for metabolite identification contains 50 metabolites detected in human serum and 48 detected in CSF.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Basic            | Web Based                             | Web User Interface                               | Unix/Linux, Mac OS, Windows                           | -                                                 | -                      | -                                                                                                                                                  | -                                                                                                | Bruker, Agilent/Varian                                                                                                     | 2015             | 2015               |                                                                                                                                                            | http://bayesil.ca/                                                                                             | https://www.ncbi.nlm.nih.gov/pubmed/26017271                       | 23373753 |                                                                         |                                             |          |
| EasyLCMS              | This software both performs metabolite annotation and peak alignment, in a targeted fashion for LC-MS data. Metabolites are annotated from the KEGG, HMDB and PubChem databases. For annotation, multiple searches are avoided by checking the interconnectivity between databases. If internal standards are used, these can be uploaded in the table format and the quantification ion list can be saved for future use. There are three steps to data processing in EasyLCMS: raw data filtering, chromatograph construction and deconvolution (the algorithms used are from the MZmine team). The Savitzky-Golay smoothing filter can be applied for raw data filtering. A centroid algorithm is used for peak detection, with all data points above a specified noise level being detected. Continuous chromatographs are then constructed. Chromatogaphs are then deconvoluted (divided into individual peaks) using an algorithm which detects local minima in the chromatograph as border points between peaks. Different types of calibration curves (linear, logarithmic, power, exponential, quadratic polynomial and cubic polynomial) can then be used for calibration between peak area and concentration (the calibration curve). For calibration to be performed standard samples at known concentrations must be used. Peak candidates are suggested using the RANSAC algorithm. Data can then be visualised and results exported in the .CSV format.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Targeted                                | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | Microsoft. NET, Microsoft Silverlight 4.0         | 1                      | Silverlight (for Mac OS and Windows)                                                                                                               | mzML, mzXML, netCDF                                                                              | -                                                                                                                          | 2012             | 2012               | Free for academic users. For non-academic users a license is required.                                                                                     | http://www.easylcms.es/                                                                                        | http://www.ncbi.nlm.nih.gov/pubmed/22884039                        | 19622150 |                                                                         |                                             |          |
| eRah                  | eRah is a CRAN based package, which provides five processing steps: data preprocessing, spectral deconvolution, spectral alignment, missing compound recovery and compound identification. During preprocessing, baseline drift is removed using a moving-minimum filter and a Savitzky Golay filter is used to eliminate noise. Compound match by local covariance (CMLC), a multivariate matched filter and orthogonal signal deconvolution (OSD) are then used to extract compound spectra. The quantitative compound profile is obtained using a least absolute deviation (LAD) regression. Compounds are clustered by retention time distance and spectral similarity for retention time alignment. This method does not require retention indices for alignment, although they can be used to increase compound identification confidence. It is possible that the algorithm will fail to group the same compound in all samples, so a missing compound recovery step is used. For identification, eRah integrates ~500 compounds from MassBank or users can import their own libraries (not NIST).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | GC-MS                          | GC-TOF/MS GC-qTOF/MS              |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.0.4                  | R (ł 2.10), Rcpp                                                                                                                                   | mzXML, netCDF                                                                                    | -                                                                                                                          | 2016             | 2016               | GPL-2 | GPL-3                                                                                                                                              | https://cran.r-project.org/web/packages/erah/index.html                                                        | http://www.ncbi.nlm.nih.gov/pubmed/27584001                        | 21401061 |                                                                         |                                             |          |
| LIMSA                 | Lipid Mass Spectrum Analysis (LIMSA) provides deconvolution, metabolite identification and quantification of lipidomics LC-MS data. Firstly background subtraction is performed (the spectra are divided into equal segments, a median filter is used and the lowest intensity of each segment is found and linear regression is calculated through the lowest points). Next the raw data are smoothed with a Gaussian filter and the local maximums are selected as candidate peak positions and local minimums as peak limits and the peaks are integrated between these limits. The user then specifies a compound list from the internal database, which contains >3000 lipids along with common fragments/adducts and lipids are identified. The user can add sum formula to the database and additional isotopes to the isotope database. There are three methods of deconvolution available: subtraction, linear fit and a Gaussian peak model. It is available as an Excel add-in or a standalone command line program.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | LC-MSDI-MS                     | DI-MS/MS                          | Lipidomics                              | Medium           | Excel Add-in/ Package                 | Command line interface, Graphical user interface | Unix/Linux, Windows                                   | C++                                               | -                      | -                                                                                                                                                  | Peaklist                                                                                         | -                                                                                                                          | 2006             | 2006               | GPL                                                                                                                                                        | http://www.helsinki.fi/science/lipids/software.html                                                            | http://www.ncbi.nlm.nih.gov/pubmed/17165823                        | 20650010 |                                                                         |                                             |          |
| LipidXplorer          | This provides alignment, annotation and metabolite identification of DI-MS lipidomics data, gathered in any mode from any tandem mass spectrometer. Firstly individual scans are merged into representative spectra. Peaks are then aligned and the signal intensity weighted average of a clustered group of peaks is used to substitute their masses. Peak intensities are stored in MasterScan. Precursor and fragment ions can serve as 'signatures' leading to lipid identification. The molecular fragment query language (MFQL) is used for searching for lipid annotations, typically searching one lipid class at a time.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | DI-MS                          | DI-MS/MS                          | Lipidomics                              | Advanced         | Python Package                        | Command line interface                           | Windows                                               | Python                                            | 1.2.7                  | numPy, lxml                                                                                                                                        | mzML, mzXML, Peaklist in csv/dta                                                                 | AB Sciex .wiff, Thermo Scientific .raw                                                                                     | 2012             | 2016               | GPL-2                                                                                                                                                      | https://wiki.mpi-cbg.de/lipidx/Main_Page                                                                       | http://www.ncbi.nlm.nih.gov/pubmed/22272252                        |          |                                                                         |                                             |          |
| MMSAT                 | Metabolite Mass Spectrometry Analysis Tool (MMSAT) is designed for the automated quantification of selected reaction monitoring (SRM) data, which is acquired from QQQ mass spectrometers. First the transitions present, based on the scans, are determined. XIC are then constructed for each transition. For each XIC, peak detection is then performed, using smoothing and optionally denoising using Savitzky-Golay smoothing and wavelet shrinkage denoising. All maxima above a user-defined intensity threshold are then found. The base of each peak is then found and the AUC is calculated. Peaks are then aligned across samples using an iterative pairwise comparison. Whether 2 peaks come from the same metabolite is determined by whether they come from the same transition and their retention time difference must be below a user-defined threshold.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     | GC-MS SRMLC-MS SRM                | Targeted                                | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | C++, python                                       | 1.1                    | -                                                                                                                                                  | .mzXML                                                                                           | -                                                                                                                          | 2011             | 2011               | Non-commerical                                                                                                                                             | http://powcs.med.unsw.edu.au/research/adult-cancer-program/services-resources/mmsat                            | http://www.ncbi.nlm.nih.gov/pubmed/22111688                        | 15529173 |                                                                         |                                             |          |
| MSeasy                | This CRAN package performs both preprocessing and metabolite identification of GC-MS data. The intensity of each fragment is transformed into a relative percentage of the highest mass fragment per spectrum. Unsupervised clustering methods are then used to group fragments. There is optionally an algorithm to compute the best clustering method for the data. When the data is clustered, it can be identified if the user provides a list of commercial standards or manually assigned reference compounds. NIST can also be directly searched for metabolite identification.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 5.3.3                  | amap, clValid, cluster, fpc, mzR, xcms                                                                                                             | mzXML, netCDF, ASCII                                                                             | Agilent .d                                                                                                                 | 2012             | 2013               | GPL-2                                                                                                                                                      | https://cran.r-project.org/web/packages/MSeasy/index.html                                                      | http://www.ncbi.nlm.nih.gov/pubmed/22782550                        | -        |                                                                         |                                             |          |
| mzMatch               | mzMatch.R is designed for LC-MS preprocessing and has integration with XCMS and the PeakML file format (which is designed to contain data from multiple chromatographic coupled MS runs). Initially features are detected in an xcmsSet with the centWave algorithm. The extracted peaks can then be visualised in PeakML files. Next biological/technical replicates are combined into a single sample. This stage includes the ability to filter by noise or RSD. Data from the sample experimental condition are then combined to allow for comparison between conditions and further filtering if desired. Retention time correction is provided by XCMS. There is also a blank filter tool that compares signal intensities between blanks and experimental samples. If the intensity of a feature is equal to or greater in a blank sample than an experimental sample, it is removed. Missing peaks can then be filled. Peaks that are not detected in at least 6 samples can then be filtered with the simple filter function. A dilution trend filter can also be applied. Groups of related peaks can then be clustered (isotopic peaks, fragments, adducts and multiply charged molecules). Peaks can then be annotated using user defined lists of metabolites. It also provides the functionality of isotopic labelling analysis. Peaks must be extracted from  LC-MS raw data using XCMS and then alignment, normalisation and filling in missing peaks must be performed with mzMatch.R, to produce a combined PeakML file. For targeted profiling a user defined, tab separated file of compounds of interest must also be provided (this list can also contain adducts and/or fragments). For untargeted profiling either all peaks can be searched for isotopes, databases such as KEGG or HMDB can be used to search for compounds or common biotransformations can be used to search for isotopes. For both isotopic profiling techniques, two files are produced a PDF and a tab-separated text file. Each page of the PDF contains an identified compound, along with its polarity, mass, etc., as well as graphs detailing each chromatograph and labelling patterns. MetAssign is a probabilistic Putative Metabolite Annotation algorithm implemented in mzMatch that assigns posterior probabilities to the likelihood of the annotation. A Bayesian Markov Chain Monte Carlo sampling scheme is used to simultaneously group features and assign molecular formulas to the groups. Within a group, an individual peak is assigned an isotope-adduct pair. The cluster model is a mixture model with a Dirichlet process prior, so that the number of different metabolites does not need to be assigned as a prior. To be clustered features must have similar retention times, masses explainable by a calculated isotope/ adduct and the correct intensity relationships. Following each iteration of the Gibbs sampling scheme the posterior probability of the annotation is calculated. The more peaks that are assigned to a formula, the more likely that it is the correct annotation.                                                         | Preprocessing, Annotation                                                                                                                                                                                                                                                                                                                           | Level 3 - Tentative Candidates                                                                                                                                                                                                                                                                 | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R, Java                                           | 2.0-13                 | R (ł 2.15.3), xcms, multtest, mzR, rJava, XML, snow, caTools, bitops, ptw, gplots, tcltk2, JRE ł 7                                                 | mzData, mzML, mzXML                                                                              | -                                                                                                                          | 2011             | 2015               | LGPL-2                                                                                                                                                     | http://mzmatch.sourceforge.net/index.php                                                                       | http://www.ncbi.nlm.nih.gov/pubmed/21401061                        | 23456039 |                                                                         |                                             |          |
| LipidQA               | Lipid Qualitative/Quantitative Analysis (LipidQA) provides smoothing, noise reduction, deconvolution, deisotoping, normalisation and metabolite identification of DI-MS lipidomics data. Peak lists are loaded and then can optionally undergo 5-point triangular smoothing and noise filtering. Next the data are deconvoluted and deisotoped. MS/MS data are searched against a fragment ion database (existing within the website) for identification. If a m/z value does not match any recorded MS/MS spectra, a lipid chemical formula database is searched and identified formulas are used to generate theoretical isotope distributions for deisotoping. Finally there are two options for quantification: Absolute Quantification and Lipid Phosphorus Normalisation. In absolute quantification, relative intensity values are compared to a calibration curve for quantification. For lipid phosphorus normalisation, the lipid phosphorus content must be specified and the quantify of each lipid species is normalised to this value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Preprocessing, Post-processing, Annotation                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                | DI-MS                          | DI-MS/MS                          | Lipidomics                              | Advanced         | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | C++                                               | -                      | -                                                                                                                                                  | mzXML                                                                                            | Finnigan .raw                                                                                                              | 2007             | 2014               |                                                                                                                                                            | http://lipidqa.dom.wustl.edu/                                                                                  | http://www.ncbi.nlm.nih.gov/pubmed/17720531                        | 24443383 |                                                                         |                                             |          |
| Automics              | This program is designed for both NMR data preprocessing and chemometric analysis. It includes modules for spectral processing, data organisation, data filtering, data analysis techniques and statistical total correlation spectroscopy method (STOCSY). It is able to read raw Bruker files and converts raw Varian and JEOL files into the Bruker FID format using a manually imputed metadata file containing parameter information. Spectral processing can be performed either manually or automatically using the following pipeline: fast fourier transformation (FFT), phase correction, baseline correction, peak alignment and bucket/binning. Using FFT NMR signals are converted from time domain to frequency domain. There are options for either complex FFT or real FFT and DC offset and zero filling can also be performed at this stage. There are two global methods for phase correction: maximising the spectrum integral and minimising the spectrum entropy, as well as an easier to implement automatic method to calculate the zero order and first order phases. There are two method for baseline correction: linear fitting and non-parametric recognition (a variant of Sergey's algorithm). Peak alignment is implemented as a fuzzy wrapping method. There are two options for bucket/binning: full resolution bucket/binning and traditional bucket/binning. There are four normalisation options: against total spectral area, against inner reference peak area, against maximal peak area and against a specific peak area. There are also four data filtering techniques: multiple signal correction (MSC), standard normal variate transform (SNV), direct orthogonal signal correction (DOSC) and orthogonal projections to latent structures (O-PLS). There are nine different data analysis techniques included within Automics: Fisher's criterion (FC), PCA, linear discriminant analysis (LDA), uncorrelated linear discriminant analysis (ULDA), K-means clustering, PLS, SVM, K-NN and soft independent modeling of class analogy (SIMCA). Three scaling methods are also available: auto scaling, central scaling and pareto scaling.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Preprocessing, Post-Processing, Statistical Analysis                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Advanced         | Package                               | Command line interface                           | Windows NT, Windows 2000-2003, Windows XP             | Visual C++                                        | 1220                   | -                                                                                                                                                  | -                                                                                                | Bruker FID                                                                                                                 | 2009             | 2009               | GPL-2                                                                                                                                                      | http://www.softpedia.com/get/Science-CAD/Automics.shtml                                                        | http://www.ncbi.nlm.nih.gov/pubmed/19291281                        | 21325300 |                                                                         |                                             |          |
| Ionwinze              | This provides preprocessing, pairwise comparison between 2 experimental conditions and statistical analysis. Positive and negative modes are analysed separately. First peaks are extracted and the data are then linearly normalised to the range 2^16 -1. The spectra of different classes are divided into equally sized segments of 1 m/z and 12 seconds RT. From these the mean ion currents and sd are obtained. The Z-factor is then calculated for each segment (z = 1 - 3(sd1 - sd2)/|mean1 - mean2|). The Z-factor is designed to be able to distinguish between conditions without the need of further statistical analysis. Univariate statistical analysis to determine significant differences between classes is also provided: coefficient of variation (CV), Student's t-test and z-test.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Medium           | Package                               | Command line interface                           | Windows (32 bit)                                      | C++                                               | 3.9.3.2                | -                                                                                                                                                  | ASCII, netCDF                                                                                    | -                                                                                                                          | 2013             | 2016               | GPL-3                                                                                                                                                      | https://sourceforge.net/projects/ionwinze/                                                                     | http://www.ncbi.nlm.nih.gov/pubmed/24004415                        | 26498454 |                                                                         |                                             |          |
| MetabolAnalyze        | This CRAN package provides probabilistic PCA (PPCA), probabilistic principal components and covariates analysis (PPCCA) and mixtures of probabilistic principal component analysis (MPPCA) for modelling metabolomics data. PPCA is based on the Gaussian latent variable model. PPCCA incorporates covariates into the model, which is useful for analysing human data, where other phenotypic data (age, gender, BMI) are also gathered, as well as metabolomic data. In this model it is assumed that the latent variable distribution for observations follows a multivariate Gaussian distribution that is not centred at the origin, like PPCA. A collection of PPCA models can be combined for MPPCA for non-linear modelling purposes. MPCCA clusters observations into groups and reduces data dimensionality.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                | MSNMR                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.3                    | mclust, mvtnorm, ellipse, gtools, gplots                                                                                                           | Peaklist matrix                                                                                  | -                                                                                                                          | 2010             | 2015               | GPL-2                                                                                                                                                      | https://cran.r-project.org/web/packages/MetabolAnalyze/index.html                                              | https://www.ncbi.nlm.nih.gov/pubmed/21092268                       | 24927477 |                                                                         |                                             |          |
| MetaboLyzer           | This software aims to identify features which have significantly different signal intensities between two treatment conditions (only 2 conditions can be analysed). First features that are not present in user-defined percentage of samples of one of the conditions are filtered (removal of missing values). Data are then segmented into two categories: partial presence (data only above threshold for one condition) and complete presence (data above threshold in both conditions). Data can then be transformed (log or inverse hyperbolic sine) and data which are log transformed then have the option of being gaussian normalised. Data with partial presence are then analysed with Fisher's exact test and data with complete presence are analysed with a variety of statistical techniques: Student's t test, Welch's t test, Mann-Whitney U test, Kolmogorov-Smirnov test, Anderson-Darling test for normality, confidence intervals, fold-change, mean and standard deviation. A volcano plot is then produced and multivariate analysis can then be performed (PCA, multidimensional scaling (MDS), kernel PCA and independent component analysis (ICA)). Lastly a heatmap is constructed. Features are putatively identified using KEGG, BioCyc, LipidMaps and HMDB. A correlation analysis can be performed after the initial analysis is completed (correlation matrices of the two conditions are first created, followed by hierarchical clustering, then Fisher's transformations coupled to Z test are used to evaluate significance and a correlational heatmap is produced).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                |                                |                                   |                                         | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS                                    | Python, R                                         | 1                      | scipy, numpy, matplotlib, rpy2, lxml, gplots, kernlab, MASS, fastICA, ICSNP, mvnormtest                                                            | Peaklist                                                                                         | -                                                                                                                          | 2013             | 2013               |                                                                                                                                                            | https://sites.google.com/a/georgetown.edu/fornace-lab-informatics/home/metabolyzer                             | http://www.ncbi.nlm.nih.gov/pubmed/24266674                        | 24168717 |                                                                         |                                             |          |
| metaXCMS              | Designed for use in conjunction with XCMS and performs meta (second order) analysis for data reduction. Once data is loaded, the control group for each experiment is verified. After this features are filtered by either fold change, p-value or predefined list of up and down regulation. Features from separate experiments are then aligned by m/z and rt using the 'group.nearest' algorithm from XCMS. Any number of experiments can be compared using this method, but only 5 or less can be visualised using a VENN diagram, showing common features. Data then re-undergo retention time correction, using the OBI-Warp method, to verify results. metaXCMS is currently only available online as part of the XCMS Online software.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | Package                               | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 0.1.20                 | GTK+, cairoDevice, RGtk2, gstat, RANN, xlsx, xcms, Biobase                                                                                         | XCMS output                                                                                      | -                                                                                                                          | 2010             | 2012               |                                                                                                                                                            | https://xcmsonline.scripps.edu/landing_page.php?pgcontent=mainPage                                             | http://www.ncbi.nlm.nih.gov/pubmed/22343432                        |          |                                                                         |                                             |          |
| MetStaT               | MetStaT is a CRAN package which provides functions for statistical analysis. Methods it provides includes: ANOVA-simultaneous component analysis (ASCA), figure of merit (FoM), PCA, GoemanŐs global test for metabolomic pathways (Q-stat), Penalized Jacobian method (for calculating network connections), Time-lagged correlation method and zero slopes method. It also includes centering and scaling functions.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                |                                |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1                      | MASS, abind, pls                                                                                                                                   | matrix                                                                                           | -                                                                                                                          | 2013             | 2013               | Apache License 2.0                                                                                                                                         | https://cran.r-project.org/web/packages/MetStaT/index.html                                                     | -                                                                  | -        |                                                                         |                                             |          |
| mQTL.NMR              | mQTL.NMR, a bioconductor package, provides a complete mQTL analysis pipeline for 1H NMR data, including processing, statistical analysis between metabolites and QTL and visualisation. NMR data should be preprocessed before being loaded into mQTL.NMR (FT, apodization, calibration, phasing and baseline correction). A number of normalisation approaches are provided: constant sum, constant noise, linear baseline and probabilistic quotient normalisation. Two scaling options are available: Pareto and autoscaling. Recursive segment wise peak alignment (RSPA) is used for peak alignment. There are two approaches to reduce dimensionality for mapping metabolites to QTLs: bucketing and statistical recoupling of variables (SRV). For animal crosses, the extended Haley-Knott regression method is used. For human mapping, simple linear regression is used to map SNPs to metabolites. Other covariants such as sex and age can also be taken into account in the model, due to the use of GenABEL.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.8.0                  | R ł 2.15.0, qtl, GenABEL                                                                                                                           | Text file                                                                                        | -                                                                                                                          | 2015             | 2016               | Artistic-2.0                                                                                                                                               | https://www.bioconductor.org/packages/release/bioc/html/mQTL.NMR.html                                          | http://www.ncbi.nlm.nih.gov/pubmed/25803548                        | 23303751 |                                                                         |                                             |          |
| RFmarkerDetector      | RFmarkerDetector is a CRAN package that provides random forest analysis of metabolomics data, as well as scaling and PCA. Scaling can be performed using either autoscaling or pareto scaling. Mean centering can also be performed. There are functions for selecting the best RF model. Cross validation is implemented with the AUC multiple cross-validation (AUCRF) as well as Monte Carlo cross validation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                |                                |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.0.1                  | AUCRF                                                                                                                                              | Peaklist                                                                                         | -                                                                                                                          | 2016             | 2016               | GPL-3                                                                                                                                                      | https://cran.r-project.org/web/packages/RFmarkerDetector/index.html                                            | -                                                                  | 19821464 |                                                                         |                                             |          |
| ropls                 | This bioconductor package implements multivariate statistical analysis approaches: PCA, PLS, PLS-DA, OPLS and OPLS-DA. It includes the original nonlinear iterative partial least squares (NIPALS) versions of the algorithms, as opposed to those implemented in muma.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Statistical Analysis                                                                                                                                                                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                |                                |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.6.2                  | -                                                                                                                                                  | data frame or matrix Peaklist                                                                    | -                                                                                                                          | 2015             | 2016               | CeCILL                                                                                                                                                     | http://bioconductor.org/packages/release/bioc/html/ropls.html                                                  | http://www.ncbi.nlm.nih.gov/pubmed/26088811                        | 26088811 |                                                                         |                                             |          |
| eMZed                 | This provides a library of modules that can be incorporated together as a workflow. The MetaboFeatureFinder (OpenMS) and matched filter and centWave (XCMS) peak picking algorithms can be used for feature detection (eMZed builds a bridge to R). EICs can be visualised. Once peaks are detected they can be integrated using the EMG integrator. Retention time alignment is then performed using Pose Clustering alignment algorithm (OpenMS). Isotope patterns are identified and compounds are identified from PubChem. Two datasets can be compared using one-way ANOVA or Kruskal Wallis statistical tests.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | Framework                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | Python                                            | 2.29.2.0               | Python 2.7, numpy, scipy, matplotlib, PyQT4, virtualenv, cython                                                                                    | mzData, mzML, mzXML                                                                              | -                                                                                                                          | 2013             | 2016               | GPL-2                                                                                                                                                      | http://emzed.ethz.ch/index.html                                                                                | http://www.ncbi.nlm.nih.gov/pubmed/23418185                        | 26283310 |                                                                         |                                             |          |
| Galaxy-M              | These are metabolomics tools for Galaxy. It includes Galaxy installations of MI-PACK, XCMS and CAMERA. PLS-Toolbox is a commercial MATLAB toolbox for chemometric multivariate analysis that can also be optionally installed. Galaxy-M is designed for use in the analysis of LC-MS and DI-MS metabolomics data. Functions from XCMS can be used for peak picking and retention time alignment. CAMERA is used for adduct annotation. A number imputation techniques are included: small value replacement, mean replacement, median replacement, KNN imputation (SimStitch) and BPCA imputation. Either TIC or PQN normalisation can be applied. Features and samples can be filtered by the number of missing values they contain. Noise filling can be applied. General logarithm transformation is also available. Features below a user-specified RSD threshold across QC samples can be removed for quality assurance. Batch correction and peak outlier detection can also be applied. PCA and univariate (2-tailed t-test/1-way ANOVA) statistical analysis is provided. MI-PACK provides empirical formula identification.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MSDI-MS                     |                                   |                                         | Medium           | Packages                              | Galaxy-based                                     | Unix/Linux, Mac OS, Windows                           | R, Python, MATLAB                                 | 1                      | Python 2.7, numpy 1.8.1, scipy 0.14.0, comptypes 0.6.1, MSFileReader 3.0, MI-PACK, Galaxy, MATLAB Complier Runtime, R (ł 3.1), Hmisc, XCMS, CAMERA | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2015             | 2016               | GPL-3                                                                                                                                                      | https://GitHub.com/Viant-Metabolomics/Galaxy-M                                                                 | https://www.ncbi.nlm.nih.gov/pubmed/26913198                       | 19015140 |                                                                         |                                             |          |
| MAIT                  | Metabolite Automatic Identification Toolkit (MAIT) is a Bioconductor package which uses XCMS for peak detection (matchedFilter or centWave), alignment and filling in missing peaks of LC-MS data. CAMERA is then used for adduct and fragment annotation. At this stage, statistical analysis is then performed (univariate statistical analysis must be performed before the next function will run). If there are two classes of data Student's t-test, Welch's t-test or Mann-Whitney tests can be used; if there are more than two classes ANOVA or Kruskal-Wallis tests can be used. There is also support for adding user defined tests e.g. Fisher's exact test. Multiple testing corrections are also provided, including Bonferroni and FDR. At this stage there is also the option to use peak aggregation techniques that may lead to better feature detection (Fernandez-Albert et al. 2014), through commercial package pagR which provides options such as PCA. 2D- and 3D- PCA plots, heatmaps and boxplots can also be produced. Biotransformation (mass losses) are then detected using a mass allowance window inside the peak group (Breitling et al. 2006). Human biotransformations are already present in MAIT, but user defined biotransformations can also be included. The HMDB (07/09) version is then used for metabolite annotation. PLS-DA and SVM with Kernel and K-NN are provided as validation techniques, which are designed to be used after the rest of the workflow has been completed. Throughout the workflow MAIT class objects are used, improving traceability.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.8.0                  | R (ł 2.10.0), CAMERA, Rcpp, pls                                                                                                                    | mzData, netCDF                                                                                   | -                                                                                                                          | 2014             | 2016               | GPL-2.0                                                                                                                                                    | https://www.bioconductor.org/packages/release/bioc/html/MAIT.html                                              | http://www.ncbi.nlm.nih.gov/pubmed/24642061                        | -        |                                                                         |                                             |          |
| Mass++                | Mass++ provides preprocessing, metabolite identification, statistical analysis and direct upload to MassBank. It is designed for both metabolomics and proteomics LC-MS data, so some of its functions are specialised for proteomics, not metabolomics. It includes 4 peak picking algorithms: MWD (for identification), GION (for quantification), AB3D (for detecting peaks from 2D data and is suitable for label-free quantification) and local maximum (simple peak-detection algorithm). Smoothing and retention time alignment (a dynamic programming algorithm) are also provided. Five different normalisation methods are provided: internal standard, integration, distribution (max intensity), distribution (median) and distribution (percentile). MassBank can be accessed directly through its SOAP API, allowing for both metabolite identification and direct upload of data to MassBank. Users can also create a private MassBank library. Users experienced with programming can create their own functions in C++ or .NET using the Mass++ standard development kit (SDK). IronPython scripts can be implemented using Mass++'s script console.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   |                                         | Medium           | Plug-in                               | Graphical user interface                         | Windows XP/Windows 7                                  | C++, C#, .NET                                     | 2.7.5                  | -                                                                                                                                                  | mzML, mzXML, netCDF                                                                              | AB Sciex .wiff, Aglient .d, KRATOS .run, Shimadzu .lcd, Shimadzu .qgd, Thermo .raw, Waters .raw                            | 2011             | 2015               | Three-clause BSD license                                                                                                                                   | http://www.shimadzu.co.jp/aboutus/ms_r/archive.html                                                            | https://www.ncbi.nlm.nih.gov/pubmed/24965016                       | 26853041 |                                                                         |                                             |          |
| MassCascade           | MassCascade is implemented in the KNIME workflow environment or alternatively can be accessed via the command line. It provides a modular pipeline for the analysis of LC-MS^n data. Data should be in the centroid format or they can be centroided with the implemented wavelet-based centroider. Firstly, a mass trace is built. The background is the subtracted and noise is reduced. At this stage the user can visualise the scans/ TIC. The data can then be normalised and filtered. Retention time alignment is performed using the obiwarp algorithm. For deconvolution Biemann and Savitzky Golay deconvolution are offered. Features can be compiled using Biemann or cosine compilers. A number of different filters are provided: Durbin, Feature and Mass. Two methods of smoothing are provided: Savitzky Golay smoother and Top Hat. For Putative Metabolite Annotation ChemSpider, MassBank and Metlin databases can be searched. Adducts can also be automatically annotated.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MS                          | LC-MS/MSLC-MS^n                   |                                         | Medium           | Package                               | Command line interface/ Graphical user interface | Unix/Linux, Mac OS, Windows (not Mac OS for KNIME)    | Java                                              | 1                      | -                                                                                                                                                  | mzML                                                                                             | Thermo Scientific .raw                                                                                                     | 2013             | 2014               | GPL-3                                                                                                                                                      | https://bitbucket.org/sbeisken/masscascadeknime/wiki/Home                                                      | https://www.ncbi.nlm.nih.gov/pubmed/26279687                       | 17263323 |                                                                         |                                             |          |
| MAVEN                 | Metabolomics and visualisation engine (MAVEN) provides a processing pipeline for LC-MS data including: preprocessing, annotation and metabolite identification. For feature detection each scan is sequentially searched for consecutive scans containing the same m/z within the user set mass error, if an m/z value fulfills this requirement, then it is considered a feature. A binary search algorithm is used to extract the EIC from every retention time window. The EIC baseline is then calculated by first removing the 20% most intense peaks then Gaussian smoothing is used on the remaining peaks. Peaks are then grouped across samples: EICs for a given m/z are summed across samples and these define 'groups', which individual EICs are assigned by overlapping retention times. Retention times are then aligned using iterative fitting with a global third degree polynomial. Peak quality scores are reported to enable to user to identify high quality peaks. KEGG Compounds can be used to putatively identify features. Significant difference between the metabolites of two datasets can be identified by using fold-change calculations.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | Package                               | Gaphical user interface                          | Unix/Linux, Mac OS, Windows                           | C++                                               | 769                    | -                                                                                                                                                  | mzXML                                                                                            | -                                                                                                                          | 2008             | 2013               |                                                                                                                                                            | http://genomics-pubs.princeton.edu/mzroll/index.php                                                            | http://www.ncbi.nlm.nih.gov/pubmed/21049934                        | 20128615 |                                                                         |                                             |          |
| MetaboAnalyst         | MetaboAnalyst 3.0 is a web server. It contains eight independent modules for analysis: statistical analysis, enrichment analysis, pathway analysis, time series analysis/ two factor design, power analysis, biomarker analysis, integrated pathway analysis and other utilities (data conversion, batch effects correction and lipidomic analysis). For statistical analysis PCA, PLS-DA, OPLS-DA, sPLS-DA, t-tests, ANOVA, heatmaps, dendrograms, random forests and SVM are supplied. Metabolite set enrichment analysis (MSEA) is performed in the enrichment module. Pathway analysis integrates enrichment analysis with pathway topology analysis. It also includes visualisation for the metabolic pathways for 21 model organisms, totalling ~1600 pathways. The time series analysis/ two factor design module includes ANOVA-simultaneous component analysis (ASCA), two-way ANOVA, and empirical Bayes time-series analysis. Users can calculate the minimum number of samples required for sufficient power for statistical analysis using the power analysis module. Biomarker analysis provides ROC curve based biomarker analyses. Metabolomic and gene expression data can be combined in the integrated pathway analysis module. Only basic support is provided for the processing of raw data. It can also be downloaded and installed locally.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | GC-MSLC-MSNMR                  |                                   |                                         | Basic            | Web Based                             | Web User Interface                               | Linux, OS X, Windows (Web-based)                      | Java, R                                           | 3                      | -                                                                                                                                                  | mzData, mzXML, netCDF, Peaklist - csv, txt                                                       | -                                                                                                                          | 2009             | 2017               |                                                                                                                                                            | http://www.metaboanalyst.ca/                                                                                   | http://www.ncbi.nlm.nih.gov/pubmed/25897128                        | 20015393 |                                                                         |                                             |          |
| MetabolomeExpress     | MetabolomeExpress is designed to perform three main functions: 1) Store GC-MS metabolomics data, allowing for analysis without the user having to download the data 2) provide a GC-MS analysis pipeline and 3) store metabolite response statistics. The PeakFinder algorithm, which uses a slope-based peak detection method is used for peak picking. Next the MRSI library matching algorithm takes EIC peak lists, a reference library and RI calibration file as input for metabolite identification. In this EIC peaks are grouped by their apex RIs into 'preliminary spectra', followed by a library search on each bin. Both internal and external RI calibration are supported. Users may supply their own libraries (including from AMDIS) or use a MetabolomeExpress supplied library. For statistical analysis: PCA, HCA, Welch's t-tests and heat maps are provided.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | GC-MS                          |                                   |                                         | Basic            | Web App                               | Web user interface                               | Unix/Linux, Mac OS, Windows                           | JavaScript, MySQL, PHP, R                         | 1                      | -                                                                                                                                                  | netCDF                                                                                           | -                                                                                                                          | 2010             | 2010               | MetabolomeExpress may be used for non-commercial purposes by people or organisations who offer all of their products and services entirely free-of-charge. | https://www.metabolome-express.org/                                                                            | http://www.ncbi.nlm.nih.gov/pubmed/20626915                        | 23162086 |                                                                         |                                             |          |
| MetMSLine             | This package provides a workflow for untargeted LC-MS data, including normalisation, smoothing, biomarker selection, annotation and metabolite identification. As input this package requires a peak list. Zero-filling is first performed, then median fold-change normalisation can be applied if required. QC-based locally weighted scatterplot smoothing is then used to alleviate the effect of signal drift. Data are then log transformed and features which are found stability across pooled samples are maintained. Outliers are then removed using expansion of Hotelling's T2 distribution ellipse (samples beyond this ellipse but within first/second compound PCA plot are removed). Potential biomarkers are then identified using multivariate regression and visualised with heatmaps. Using MS/MS data, features can then be identified by matching to fragments MS/MS spectra. Metabolites can also be identified via a user-inputed .csv database of compounds and their exact masses.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   | Untargeted                              | Advanced         | R Package                             | Command line interface/ Graphical user interface | Unix/Linux, Mac OS, Windows                           | R                                                 | 1.0.1                  | R (ł 3.2.0), fgui, tcltk2, zoo, gplots, ggplot2, sfsmisc, RColorBrewer, readMzXmlData                                                              | Peaklist (xcms), covariate table (.csv)                                                          | -                                                                                                                          | 2014             | 2015               | GPL-3                                                                                                                                                      | http://wmbedmands.github.io/MetMSLine/                                                                         | http://www.ncbi.nlm.nih.gov/pubmed/25348215                        | 25527831 |                                                                         |                                             |          |
| MVAPACK               | This is designed as a complete workflow for NMR analysis. It contains functions for preprocessing, pretreatment, modeling and validation. Preprocessing functions include: apodization, zero-filling, Fourier transformation, phase correction, ROI selection, peak picking, integration and referencing. Functions for data pretreatment include scaling, normalisation, alignment, binning and denoising. A number of statistical techniques are also provided: PCA, LDA, PLS and OPLS. Monte Carlo n-fold internal cross-validation is used and CV-ANOVA and response permutation significance testing can be used to validate supervised models. It also contains an implementation of the icoshift algorithm.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | NMR                            | 1H NMR                            |                                         | Advanced         | Package                               | Command line interface                           | Linux, OS X                                           | Octave                                            |                        | -                                                                                                                                                  | NMRPipe                                                                                          | Bruker                                                                                                                     | 2014             | 2015               | GPL-3                                                                                                                                                      | http://bionmr.unl.edu/mvapack.php                                                                              | http://www.ncbi.nlm.nih.gov/pubmed/24576144                        | 24397582 |                                                                         |                                             |          |
| specmine              | This CRAN package provides tools for metabolomics preprocessing, identification and statistical analysis. Functions for preprocessing include: background correction, baseline correction, offset correction, feature selection, filtering, peak grouping, missing value imputation (K-NN, linear approximation, mean, median or value replacement), mean centering, multiplicative scatter correction, normalisation (sum, median, reference sample or reference feature), scaling (auto, range, pareto or interval) and shift correction (constant, interpolation) . Statistics provided include: ANOVA, clustering, correlation, FC analysis, heatmaps, HCA, k-means clustering, Kruskal-Wallis Test, Kolmogorov-Smirnov Test, linear regression, AUC, multifactor ANOVA, PCA and t-tests.  There are also functions for the visualisation of data including box plots, volcano plots and spectra. It can take data in the ChemoSpec and hyperspec formats and convert it for further analysis. xcmsSets can be imported for metabolite identification, using the MAIT package.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | NMRIRUV/VISRamanLC-MS GC-MS    |                                   |                                         | Advanced         | R Package                             | Command line interface                           | Unix/Linux, Mac OS, Windows                           | R                                                 | 1                      | R (ł 3.1.0)                                                                                                                                        | Matrix, .csv, .jdx, .dx, .spc                                                                    | -                                                                                                                          | 2015             | 2015               | GPL-2 | GPL-3                                                                                                                                              | https://cran.r-project.org/web/packages/specmine/index.html                                                    | http://www.ncbi.nlm.nih.gov/pubmed/26853041                        | 16448051 |                                                                         |                                             |          |
| Workflow4Metabolomics | Workflow4Metabolomics provides a comprehensive tool for metabolomics data preprocessing, statistical analysis and annotation. It is built in the galaxy environment and accessible as a web-based tool. It encompasses analysis pipelines for LC-MS, GC-MS and NMR, with LC-MS being the most comprehensive and NMR being the least. In the LC-MS workflow, data are preprocessing using XCMS (xcmsSet, group, retcor, group, fillPeaks) and annotated using CAMERA. Alternatively it is possible to upload a peak list of already preprocessed data for further analysis. Features can then be filtered and batch correction (linear or loess) can be performed. A number of univariate: Student's t test, Wilcoxon t test, ANOVA, Kruskal-Wallis test, Pearson or Spearman correlation test and multivariate: PCA, PLS, OPLS and PLS-DA (integrated as part of the ropls package). For Putative Metabolite Annotation HMDB, LipidMaps, KEGG-compounds and ChemSpider can be searched. Also MassBank can be searched. The HiRes algorithm can be used for empirical formula identification. GC-MS data is also preprocessed using XCMS and the same statistical analysis can be used. Metabolites can be identified using Golm or NIST if the user has it installed. For NMR a full pipeline is not available. Instead three functions are provided: Bruker bucketing and integration, normalisation (total intensity, quantitative variable and probabilistic quotient) and statistical analysis.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | LC-MS                          |                                   |                                         | Medium           | Package                               | Graphical User Interface                         | Unix/Linux, Mac OS, Windows                           | Galaxy                                            | 2.5.1                  | -                                                                                                                                                  | mzData, mzML, mzXML, netCDF                                                                      | -                                                                                                                          | 2014             | 2016               |                                                                                                                                                            | http://workflow4metabolomics.org/                                                                              | http://www.ncbi.nlm.nih.gov/pubmed/25527831                        | 22533540 |                                                                         |                                             |          |
| XCMS Online           | XCMS Online is an online implementation of XCMS, that include the entire data processing pipeline, from preprocessing to metabolite identification and statistical analysis. The processing pipeline is divided into three stages: data upload, parameter selection and result interpretation. Predefined parameter sets are available for different instrument setups or alternatively the user can set the parameters manually. For peak picking, grouping and retention time alignment, the algorithms implemented in XCMS are used. After a submitted job has been completed, plots of experimental results are displayed and the data are available for downloading. Retention time plots before and after alignment are displayed and a mirror plot (a plot showing dysregulated features) is used to visualise features with statistically significance differences in signal intensity between experimental conditions. Multidimensional scaling plots (MDS) and PCA plots are also included, both of which are performed on centred and scaled data (unit variance, log, pareto). Other visualisation provided includes EIC, box plots, cloud plots and Venn/Edwards diagrams. Putative Metabolite Annotation is performed using the METLIN database. Statistical analysis includes: Welch's t-test, Mann-Whitney test, paired t-test, Wilcoxon signed rank, one-way ANOVA with post-HOC, Kruskal-Wallis, repeated measures ANOVA and Friedman test. As with the other analysis, the user must input parameter settings (parametric or non-parametric and independent or dependent) and XCMS Online will automatically choose which statistical analysis to use. Features can be filtered by fold-change, ion intensity or p-value. metaXCMS is included, allowing for paired two-group analysis, multigroup analysis and meta-analysis. It also allows for pathway analysis and data integration of proteomics and transcriptomics data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Workflow                                                                                                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                                | GC-MSLC-MS                     |                                   | Untargeted                              | Basic            | Web Based                             | Web User Interface                               | Unix/Linux, Mac OS, Windows                           | R                                                 | 3.5.1                  | -                                                                                                                                                  | mzData, mzML, mzXML, netCDF                                                                      | Agilent .d, AB SCIENX .wiff, Waters .raw, Thermo Fisher Scientific .raw, Bruker .d                                         | 2012             | 2016               |                                                                                                                                                            | https://xcmsonline.scripps.edu/landing_page.php?pgcontent=mainPage                                             | http://www.ncbi.nlm.nih.gov/pubmed/22533540                        | 23323971 |                                                                         |                                             |          |
