# Integrated Interactome System
Version: 1.0.2

## Website
http://bioinfo03.ibi.unicamp.br/lnbio/IIS2/

## Description
Integrated Interactome System (IIS) is a web-based tool that allows integration of different bioinformatics software for the analysis of multi-omics datasets. It designed primarily for the entire processing pipeline of yeast two-hybrid system screens (Y2H), with modules available for processing, annotation, analysis and visualisation. It supports the analysis of preprocessed metabolomics data, uploaded as a peaklist with either HMDB, YMDB or ECMD IDs. Proteins/genes can be searched against 9 databases for identification, however this is not possible for metabolomics data. A database of interactions (Global Protein-Metabolite-Gene-Drug Interaction Database (GPMCDID)) can be searched to build networks. This database is constructed from nine databases including HMDB, YMDB, ECMDB and DrugBank. Networks can be built and visualised in cytoscape. KEGG database enrichment analysis can be performed.

## Functionality
- Pathway Analysis

## Instrument Data Type
- Multi-omics
- Y2H
- Proteomics
- Transcriptomics
- Metabolomics

## Approaches
-

## Computer Skills
Web App

## Software Type
- Web Based

## Interface
- Web User Interface

## Operating System (OS)
- Unix/Linux
- Mac OS
- Windows

## Language
JavaScript, JSON, PHP

## Dependencies
N/A

## Input Formats - Open
txt Peaklist

## Input Formats - Proprietary

## Published
2014

## Last Updated
2014

## License

## Paper
https://www.ncbi.nlm.nih.gov/pubmed/24949626

## PMID
24949626
